<!doctype html><html lang=en-us><head><meta charset=utf-8><meta name=viewport content="width=device-width,initial-scale=1"><meta http-equiv=x-ua-compatible content="IE=edge"><meta name=generator content="Wowchemy 4.8.0 for Hugo"><meta name=description content="Open Neuroscience is a volunteer organized project to aggregate and curate open source projects related to Neurosciences"><link rel=alternate hreflang=en-us href=https://amchagas.github.io/open-source-toolkit/><link rel=preconnect href=https://fonts.gstatic.com crossorigin><meta name=theme-color content="#2962ff"><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.14.0/css/all.min.css integrity="sha256-FMvZuGapsJLjouA6k7Eo2lusoAX9i0ShlWFG6qt7SLc=" crossorigin=anonymous><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.5.7/jquery.fancybox.min.css integrity="sha256-Vzbj7sDDS/woiFS3uNKo8eIuni59rjyNGtXfstRzStA=" crossorigin=anonymous><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.2.0/styles/github.min.css crossorigin=anonymous title=hl-light><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.2.0/styles/dracula.min.css crossorigin=anonymous title=hl-dark disabled><script src=https://cdnjs.cloudflare.com/ajax/libs/lazysizes/5.2.2/lazysizes.min.js integrity="sha512-TmDwFLhg3UA4ZG0Eb4MIyT1O1Mb+Oww5kFG0uHqXsdbyZz9DcvYQhKpGgNkamAI6h2lGGZq2X8ftOJvF/XjTUg==" crossorigin=anonymous async></script><link rel=stylesheet href="https://fonts.googleapis.com/css?family=Montserrat:400,700%7CRoboto:400,400italic,700%7CRoboto+Mono&display=swap"><link rel=stylesheet href=/open-source-toolkit/css/wowchemy.css><link rel=alternate href=/open-source-toolkit/index.xml type=application/rss+xml title="Open Source Toolkit"><link rel=manifest href=/open-source-toolkit/index.webmanifest><link rel=icon type=image/png href=/open-source-toolkit/images/icon_hu0b7a4cb9992c9ac0e91bd28ffd38dd00_9727_32x32_fill_lanczos_center_2.png><link rel=apple-touch-icon type=image/png href=/open-source-toolkit/images/icon_hu0b7a4cb9992c9ac0e91bd28ffd38dd00_9727_192x192_fill_lanczos_center_2.png><link rel=canonical href=https://amchagas.github.io/open-source-toolkit/><meta property="twitter:card" content="summary_large_image"><meta property="twitter:site" content="@openneurosci"><meta property="twitter:creator" content="@openneurosci"><meta property="og:site_name" content="Open Source Toolkit"><meta property="og:url" content="https://amchagas.github.io/open-source-toolkit/"><meta property="og:title" content="Open Source Toolkit"><meta property="og:description" content="Open Neuroscience is a volunteer organized project to aggregate and curate open source projects related to Neurosciences"><meta property="og:image" content="https://amchagas.github.io/open-source-toolkit/media/openneuroscience_logo_dark.svg"><meta property="twitter:image" content="https://amchagas.github.io/open-source-toolkit/media/openneuroscience_logo_dark.svg"><meta property="og:locale" content="en-us"><meta property="og:updated_time" content="2030-06-01T13:00:00+00:00"><script type=application/ld+json>{"@context":"https://schema.org","@type":"WebSite","potentialAction":{"@type":"SearchAction","target":"https://amchagas.github.io/open-source-toolkit/?q={search_term_string}","query-input":"required name=search_term_string"},"url":"https://amchagas.github.io/open-source-toolkit/"}</script><script type=application/ld+json>{"@context":"https://schema.org","@type":"Organization","@id":"https://amchagas.github.io/open-source-toolkit/","name":"Open Source Toolkit","logo":"https://amchagas.github.io/open-source-toolkit/images/icon_hu0b7a4cb9992c9ac0e91bd28ffd38dd00_9727_192x192_fill_lanczos_center_2.png","image":"https://amchagas.github.io/open-source-toolkit/media/openneuroscience_logo_dark.svg","url":"https://amchagas.github.io/open-source-toolkit/"}</script><head><link rel=stylesheet href=applause-button.css><script src=applause-button.js></script></head><title>Open Source Toolkit</title></head><body id=top data-spy=scroll data-offset=70 data-target=#navbar-main><script>window.wcDarkLightEnabled=true;</script><script>const isSiteThemeDark=false;</script><script src=/open-source-toolkit/js/load-theme.js></script><aside class=search-results id=search><div class=container><section class=search-header><div class="row no-gutters justify-content-between mb-3"><div class=col-6><h1>Search</h1></div><div class="col-6 col-search-close"><a class=js-search href=#><i class="fas fa-times-circle text-muted" aria-hidden=true></i></a></div></div><div id=search-box><input name=q id=search-query placeholder=Search... autocapitalize=off autocomplete=off autocorrect=off spellcheck=false type=search class=form-control></div></section><section class=section-search-results><div id=search-hits></div></section></div></aside><nav class="navbar navbar-expand-lg navbar-light compensate-for-scrollbar" id=navbar-main><div class=container><div class="d-none d-lg-inline-flex"><a class=navbar-brand href=/open-source-toolkit/>Open Source Toolkit</a></div><button type=button class=navbar-toggler data-toggle=collapse data-target=#navbar-content aria-controls=navbar aria-expanded=false aria-label="Toggle navigation">
<span><i class="fas fa-bars"></i></span></button><div class="navbar-brand-mobile-wrapper d-inline-flex d-lg-none"><a class=navbar-brand href=/open-source-toolkit/>Open Source Toolkit</a></div><div class="navbar-collapse main-menu-item collapse justify-content-center" id=navbar-content><ul class="navbar-nav d-md-inline-flex"><li class=nav-item><a class=nav-link href=/open-source-toolkit/#intro data-target=#intro><span>Home</span></a></li><li class=nav-item><a class=nav-link href=/open-source-toolkit/#topics data-target=#topics><span>Topics</span></a></li><li class=nav-item><a class=nav-link href=/open-source-toolkit/#about data-target=#about><span>About</span></a></li><li class=nav-item><a class=nav-link href=/open-source-toolkit/#people data-target=#people><span>Team</span></a></li><li class=nav-item><a class=nav-link href=/open-source-toolkit/#contact data-target=#contact><span>Contact</span></a></li></ul></div><ul class="nav-icons navbar-nav flex-row ml-auto d-flex pl-md-2"><li class=nav-item><a class="nav-link js-search" href=# aria-label=Search><i class="fas fa-search" aria-hidden=true></i></a></li><li class="nav-item dropdown theme-dropdown"><a href=# class=nav-link data-toggle=dropdown aria-haspopup=true><i class="fas fa-moon" aria-hidden=true></i></a><div class=dropdown-menu><a href=# class="dropdown-item js-set-theme-light"><span>Light</span></a>
<a href=# class="dropdown-item js-set-theme-dark"><span>Dark</span></a>
<a href=# class="dropdown-item js-set-theme-auto"><span>Automatic</span></a></div></li></ul></div></nav><span class="js-widget-page d-none"></span><section id=intro class="home-section wg-blank" style="padding:20px 0"><div class=container><div class=row><div class=col-lg-12><div align=justify><p>Thanks for visiting our page!</p><p>Open Source toolkit is a community curated page, displaying open hardware papers</p><p>Contribute by:</p><ul><li>TBA</li></ul></div></div></div></div></section><section id=topics class="home-section wg-portfolio"><div class=container><div class=margin-auto><div class=center-text><h1 class=mt-0>Topics</h1></div><div><span class="d-none default-project-filter">*</span><div class=project-toolbar><div class=project-filters><div class=btn-toolbar><div class="btn-group flex-wrap"><a href=# data-filter=* class="btn btn-primary btn-lg active">All</a>
<a href=# data-filter=.js-id-Animal-ephys class="btn btn-primary btn-lg">Animal electrophysiology</a>
<a href=# data-filter=.js-id-Topic class="btn btn-primary btn-lg">Backlog</a>
<a href=# data-filter=.js-id-Benchtop class="btn btn-primary btn-lg">Benchtop</a>
<a href=# data-filter=.js-id-Behaviour class="btn btn-primary btn-lg">Behaviour</a>
<a href=# data-filter=.js-id-Calcium-Imaging class="btn btn-primary btn-lg">Calcium Imaging</a>
<a href=# data-filter=.js-id-Computers class="btn btn-primary btn-lg">Computers</a>
<a href=# data-filter=.js-id-Computer-clusters class="btn btn-primary btn-lg">Computer Clusters</a>
<a href=# data-filter=.js-id-Data-analysis class="btn btn-primary btn-lg">Data Analysis</a>
<a href=# data-filter=.js-id-Database class="btn btn-primary btn-lg">Data Repositories</a>
<a href=# data-filter=.js-id-Electric class="btn btn-primary btn-lg">Electric Stimulation</a>
<a href=# data-filter=.js-id-Fluorescence class="btn btn-primary btn-lg">Fluorescence</a>
<a href=# data-filter=.js-id-Hardware class="btn btn-primary btn-lg">Hardware</a>
<a href=# data-filter=.js-id-Human-Neuroscience class="btn btn-primary btn-lg">Human Neuroscience</a>
<a href=# data-filter=.js-id-Human-ephys class="btn btn-primary btn-lg">Human electrophysiology</a>
<a href=# data-filter=.js-id-Microscope class="btn btn-primary btn-lg">Microscopes</a>
<a href=# data-filter=.js-id-Microscopes-software class="btn btn-primary btn-lg">Microscopes software</a>
<a href=# data-filter=.js-id-Optogenetics class="btn btn-primary btn-lg">Optogenetics</a>
<a href=# data-filter=.js-id-Prosthetics class="btn btn-primary btn-lg">Prosthetics</a>
<a href=# data-filter=.js-id-Simulation class="btn btn-primary btn-lg">Simulations</a>
<a href=# data-filter=.js-id-Software class="btn btn-primary btn-lg">Software</a>
<a href=# data-filter=.js-id-Learning class="btn btn-primary btn-lg">Tutorials and learning portals</a>
<a href=# data-filter=.js-id-Other class="btn btn-primary btn-lg">Other</a></div></div></div></div><div class="isotope projects-container js-layout-masonry"><div class="project-card project-item isotope-item js-id-Research-Article js-id-2 js-id-PLOS-Computational-Biology"><div class=card><a href=/open-source-toolkit/post/albatradis_comparative_analysis_of_large_datasets_from_parallel_transposon_mutagenesis_experiments_plos_computational_biology/ class="card-image hover-overlay"><img src=/open-source-toolkit/post/albatradis_comparative_analysis_of_large_datasets_from_parallel_transposon_mutagenesis_experiments_plos_computational_biology/featured_hud109d9c66059217af2bab44795aa78ad_52218_550x0_resize_q90_lanczos_2.png alt class=img-responsive></a><div class=card-text><h4><a href=/open-source-toolkit/post/albatradis_comparative_analysis_of_large_datasets_from_parallel_transposon_mutagenesis_experiments_plos_computational_biology/>AlbaTraDIS: Comparative analysis of large datasets from parallel transposon mutagenesis experiments</a></h4><div class=article-style><p>Publishing date: &lsquo;2020-08-18&rsquo;
Published on: &lsquo;PLOS Computational Biology&rsquo;
summary: &lsquo;In this software paper, Page et al develop ‘AlbaTraDIS’; a software application for rapid large-scale comparative analysis of TraDIS experiments that predicts the impact of transposon insertions on nearby genes.</p></div></div></div></div><div class="project-card project-item isotope-item js-id-Research-Article js-id-9 js-id-PLOS-ONE"><div class=card><a href=/open-source-toolkit/post/assessing_the_impact_of_introductory_programming_workshops_on_the_computational_reproducibility_of_biomedical_workflows_plos_one/ class="card-image hover-overlay"><img src=/open-source-toolkit/post/assessing_the_impact_of_introductory_programming_workshops_on_the_computational_reproducibility_of_biomedical_workflows_plos_one/featured_hud109d9c66059217af2bab44795aa78ad_52218_550x0_resize_q90_lanczos_2.png alt class=img-responsive></a><div class=card-text><h4><a href=/open-source-toolkit/post/assessing_the_impact_of_introductory_programming_workshops_on_the_computational_reproducibility_of_biomedical_workflows_plos_one/>Assessing the impact of introductory programming workshops on the computational reproducibility of biomedical workflows</a></h4><div class=article-style><p>Publishing date: &lsquo;2020-08-18&rsquo;
Published on: &lsquo;PLOS ONE&rsquo;
summary: &lsquo;This mixed methods study consisted of in-depth interviews with 14 biomedical researchers before and after participation in an introductory programming workshop. During the interviews, participants described their research workflows and responded to a quantitative checklist measuring reproducible behaviors.</p></div></div></div></div><div class="project-card project-item isotope-item js-id-Research-Article js-id-1 js-id-PLOS-Computational-Biology"><div class=card><a href=/open-source-toolkit/post/bigpint_a_bioconductor_visualization_package_that_makes_big_data_pintsized_plos_computational_biology/ class="card-image hover-overlay"><img src=/open-source-toolkit/post/bigpint_a_bioconductor_visualization_package_that_makes_big_data_pintsized_plos_computational_biology/featured_hud109d9c66059217af2bab44795aa78ad_52218_550x0_resize_q90_lanczos_2.png alt class=img-responsive></a><div class=card-text><h4><a href=/open-source-toolkit/post/bigpint_a_bioconductor_visualization_package_that_makes_big_data_pintsized_plos_computational_biology/>bigPint: A Bioconductor visualization package that makes big data pint-sized</a></h4><div class=article-style><p>Publishing date: &lsquo;2020-08-18&rsquo;
Published on: &lsquo;PLOS Computational Biology&rsquo;
summary: &lsquo;bigPint is a software package for interactive and static data visualisation in R. It scales comfortably to large datasets, sporting graphs that include scatterplots, violin plots, and litre plots.</p></div></div></div></div><div class="project-card project-item isotope-item js-id-Web-Article js-id-7 js-id-Futurism"><div class=card><a href=/open-source-toolkit/post/github_sealed_away_its_open_source_code_in_an_arctic_fault__futurism/ class="card-image hover-overlay"><img src=/open-source-toolkit/post/github_sealed_away_its_open_source_code_in_an_arctic_fault__futurism/featured_hud109d9c66059217af2bab44795aa78ad_52218_550x0_resize_q90_lanczos_2.png alt class=img-responsive></a><div class=card-text><h4><a href=/open-source-toolkit/post/github_sealed_away_its_open_source_code_in_an_arctic_fault__futurism/>GitHub Sealed Away Its Open Source Code In an Arctic Fault</a></h4><div class=article-style><p>Publishing date: &lsquo;2020-08-18&rsquo;
Published on: &lsquo;Futurism&rsquo;
summary: &lsquo;Earlier this month, the code management platform GitHub sealed away its archive of open source software in an Arctic vault so deep that they say it could survive a nuclear blast.</p></div></div></div></div><div class="project-card project-item isotope-item js-id-Web-Article js-id-8 js-id-Forbes"><div class=card><a href=/open-source-toolkit/post/new_opensource_software_looks_to_kickstart_the_autonomous_revolution_forbes/ class="card-image hover-overlay"><img src=/open-source-toolkit/post/new_opensource_software_looks_to_kickstart_the_autonomous_revolution_forbes/featured_hud109d9c66059217af2bab44795aa78ad_52218_550x0_resize_q90_lanczos_2.png alt class=img-responsive></a><div class=card-text><h4><a href=/open-source-toolkit/post/new_opensource_software_looks_to_kickstart_the_autonomous_revolution_forbes/>New Open-Source Software Looks To Kickstart The Autonomous Revolution</a></h4><div class=article-style><p>Publishing date: &lsquo;2020-08-18&rsquo;
Published on: &lsquo;Forbes&rsquo;
summary: &lsquo;This article reports on Project ASLAN, a collaborative alternative to the tech race at the forefront of the move towards fully-autonomous vehicles. ASLAN is a supergroup of tech companies, research groups and universities, focused on bringing a fiscally practical, fully-autonomous vehicle solution to public roads.</p></div></div></div></div><div class="project-card project-item isotope-item js-id-Research-Article js-id-3 js-id-PLOS-Computational-Biology"><div class=card><a href=/open-source-toolkit/post/ossdbs_opensource_simulation_platform_for_deep_brain_stimulation_with_a_comprehensive_automated_modeling_plos_computational_biology/ class="card-image hover-overlay"><img src=/open-source-toolkit/post/ossdbs_opensource_simulation_platform_for_deep_brain_stimulation_with_a_comprehensive_automated_modeling_plos_computational_biology/featured_hud109d9c66059217af2bab44795aa78ad_52218_550x0_resize_q90_lanczos_2.png alt class=img-responsive></a><div class=card-text><h4><a href=/open-source-toolkit/post/ossdbs_opensource_simulation_platform_for_deep_brain_stimulation_with_a_comprehensive_automated_modeling_plos_computational_biology/>OSS-DBS: Open-source simulation platform for deep brain stimulation with a comprehensive automated modeling</a></h4><div class=article-style><p>Publishing date: &lsquo;2020-08-18&rsquo;
Published on: &lsquo;PLOS Computational Biology&rsquo;
summary: &lsquo;In this study, Butenko et al propose a new open-source simulation platform that comprises computer-aided design and computer-aided engineering tools for highly automated evaluation of electric field distribution and neural activation during Deep Brain Stimulation (DBS).</p></div></div></div></div><div class="project-card project-item isotope-item js-id-Research-Article js-id-4 js-id-PLOS-ONE"><div class=card><a href=/open-source-toolkit/post/refshannon_a_genomeguided_transcriptome_assembler_using_sparse_flow_decomposition_plos_one/ class="card-image hover-overlay"><img src=/open-source-toolkit/post/refshannon_a_genomeguided_transcriptome_assembler_using_sparse_flow_decomposition_plos_one/featured_hud109d9c66059217af2bab44795aa78ad_52218_550x0_resize_q90_lanczos_2.png alt class=img-responsive></a><div class=card-text><h4><a href=/open-source-toolkit/post/refshannon_a_genomeguided_transcriptome_assembler_using_sparse_flow_decomposition_plos_one/>RefShannon: A genome-guided transcriptome assembler using sparse flow decomposition</a></h4><div class=article-style><p>Publishing date: &lsquo;2020-08-18&rsquo;
Published on: &lsquo;PLOS ONE&rsquo;
summary: &lsquo;High throughput RNA-Seq has become a staple in modern molecular biology, with applications not only in quantifying gene expression but also in isoform-level analysis of the RNA transcripts.</p></div></div></div></div><div class="project-card project-item isotope-item js-id-Research-Article js-id-5 js-id-PLOS-Computational-Biology"><div class=card><a href=/open-source-toolkit/post/the_genome_polishing_tool_polca_makes_fast_and_accurate_corrections_in_genome_assemblies_plos_computational_biology/ class="card-image hover-overlay"><img src=/open-source-toolkit/post/the_genome_polishing_tool_polca_makes_fast_and_accurate_corrections_in_genome_assemblies_plos_computational_biology/featured_hud109d9c66059217af2bab44795aa78ad_52218_550x0_resize_q90_lanczos_2.png alt class=img-responsive></a><div class=card-text><h4><a href=/open-source-toolkit/post/the_genome_polishing_tool_polca_makes_fast_and_accurate_corrections_in_genome_assemblies_plos_computational_biology/>The genome polishing tool POLCA makes fast and accurate corrections in genome assemblies</a></h4><div class=article-style><p>Publishing date: &lsquo;2020-08-18&rsquo;
Published on: &lsquo;PLOS Computational Biology&rsquo;
summary: &lsquo;In this report, Zimin and Salzberg present the assembly polishing tool POLCA (POLishing by Calling Alternatives) and compare its performance with two other popular polishing programs, Pilon and Racon.</p></div></div></div></div><div class="project-card project-item isotope-item js-id-Web-Article js-id-10 js-id-TechRepublic"><div class=card><a href=/open-source-toolkit/post/this_new_open_source_project_could_be_key_to_securing_database_applications__techrepublic/ class="card-image hover-overlay"><img src=/open-source-toolkit/post/this_new_open_source_project_could_be_key_to_securing_database_applications__techrepublic/featured_hud109d9c66059217af2bab44795aa78ad_52218_550x0_resize_q90_lanczos_2.png alt class=img-responsive></a><div class=card-text><h4><a href=/open-source-toolkit/post/this_new_open_source_project_could_be_key_to_securing_database_applications__techrepublic/>This new open source project could be key to securing database applications</a></h4><div class=article-style><p>Publishing date: &lsquo;2020-08-18&rsquo;
Published on: &lsquo;TechRepublic&rsquo;
summary: &lsquo;Moving database applications to the cloud has been a boon for development teams anxious to move faster. It has also, however, exposed security flaws inherent in traditional security solutions, something data layer security startup Cyral has been tackling.</p></div></div></div></div><div class="project-card project-item isotope-item js-id-Research-Article js-id-6 js-id-PLOS-ONE"><div class=card><a href=/open-source-toolkit/post/_development_of_a_lowcost_usercustomizable_highspeed_camera_plos_one/ class="card-image hover-overlay"><img src=/open-source-toolkit/post/_development_of_a_lowcost_usercustomizable_highspeed_camera_plos_one/featured_hud109d9c66059217af2bab44795aa78ad_52218_550x0_resize_q90_lanczos_2.png alt class=img-responsive></a><div class=card-text><h4><a href=/open-source-toolkit/post/_development_of_a_lowcost_usercustomizable_highspeed_camera_plos_one/>Development of a low-cost, user-customizable, high-speed camera</a></h4><div class=article-style><p>Publishing date: &lsquo;2020-06-15&rsquo;
Published on: &lsquo;PLOS ONE&rsquo;
summary: &lsquo;Channel Editor Summary: The paper describes a high speed camera system, capable of reaching frame rates of ~2300 frames per second in its max speed setting.</p></div></div></div></div><div class="project-card project-item isotope-item js-id-Web-Article js-id-12 js-id-Nature"><div class=card><a href=/open-source-toolkit/post/african_scientists_leverage_open_hardware_nature/ class="card-image hover-overlay"><img src=/open-source-toolkit/post/african_scientists_leverage_open_hardware_nature/featured_hud109d9c66059217af2bab44795aa78ad_52218_550x0_resize_q90_lanczos_2.png alt class=img-responsive></a><div class=card-text><h4><a href=/open-source-toolkit/post/african_scientists_leverage_open_hardware_nature/>African scientists leverage open hardware</a></h4><div class=article-style><p>Publishing date: &lsquo;2020-06-15&rsquo;
Published on: &lsquo;Nature&rsquo;
summary: &lsquo;This article looks at how a growing emphasis on do-it-yourself science is helping researchers to equip labs in resource-limited areas. For example, Actifield n automated open source actimeter for rodents developed by Victor Kumbol at the Kwame Nkrumah University of Science and Technology in Kumasi, Ghana.</p></div></div></div></div><div class="project-card project-item isotope-item js-id-Research-Article js-id-None js-id-bioRxiv"><div class=card><a href=/open-source-toolkit/post/bottomilluminated_orbital_shaker_for_microalgae_cultivation__biorxiv/ class="card-image hover-overlay"><img src=/open-source-toolkit/post/bottomilluminated_orbital_shaker_for_microalgae_cultivation__biorxiv/featured_hud109d9c66059217af2bab44795aa78ad_52218_550x0_resize_q90_lanczos_2.png alt class=img-responsive></a><div class=card-text><h4><a href=/open-source-toolkit/post/bottomilluminated_orbital_shaker_for_microalgae_cultivation__biorxiv/>Bottom-Illuminated Orbital Shaker for Microalgae Cultivation</a></h4><div class=article-style><p>Publishing date: &lsquo;2020-06-15&rsquo;
Published on: &lsquo;bioRxiv&rsquo;
summary: &lsquo;None&rsquo;
authors: &lsquo;Jakub Nedbal, Lu Gao, Klaus Suhling&rsquo;
link to paper: &lsquo;https://doi.org/10.1101/2020.05.01.071878'
Icons made by catkuro from www.flaticon.com</p></div></div></div></div><div class="project-card project-item isotope-item js-id-Research-Article js-id-None js-id-HardwareX"><div class=card><a href=/open-source-toolkit/post/diy_hybrid_ssvepp300_led_stimuli_for_bci_platform_using_emotiv_eeg_headset_hardwarex/ class="card-image hover-overlay"><img src=/open-source-toolkit/post/diy_hybrid_ssvepp300_led_stimuli_for_bci_platform_using_emotiv_eeg_headset_hardwarex/featured_hud109d9c66059217af2bab44795aa78ad_52218_550x0_resize_q90_lanczos_2.png alt class=img-responsive></a><div class=card-text><h4><a href=/open-source-toolkit/post/diy_hybrid_ssvepp300_led_stimuli_for_bci_platform_using_emotiv_eeg_headset_hardwarex/>DIY hybrid SSVEP-P300 LED stimuli for BCI platform using EMOTIV EEG headset</a></h4><div class=article-style><p>Publishing date: &lsquo;2020-06-15&rsquo;
Published on: &lsquo;HardwareX&rsquo;
summary: &lsquo;None&rsquo;
authors: &lsquo;Sure jMouli, Ramaswamy Palaniappan&rsquo;
link to paper: &lsquo;https://doi.org/10.1016/j.ohx.2020.e00113'
Icons made by catkuro from www.flaticon.com</p></div></div></div></div><div class="project-card project-item isotope-item js-id-Research-Article js-id-None js-id-HardwareX"><div class=card><a href=/open-source-toolkit/post/fishcam_a_lowcost_open_source_autonomous_camera_for_aquatic_research_hardwarex/ class="card-image hover-overlay"><img src=/open-source-toolkit/post/fishcam_a_lowcost_open_source_autonomous_camera_for_aquatic_research_hardwarex/featured_hud109d9c66059217af2bab44795aa78ad_52218_550x0_resize_q90_lanczos_2.png alt class=img-responsive></a><div class=card-text><h4><a href=/open-source-toolkit/post/fishcam_a_lowcost_open_source_autonomous_camera_for_aquatic_research_hardwarex/>FishCam: A low-cost open source autonomous camera for aquatic research</a></h4><div class=article-style><p>Publishing date: &lsquo;2020-06-15&rsquo;
Published on: &lsquo;HardwareX&rsquo;
summary: &lsquo;In this work, Mouy et al describe the “FishCam”, a low-cost (500 USD) autonomous camera package to record videos and images underwater. The system is composed of easily accessible components and can be programmed to turn ON and OFF on customizable schedules.</p></div></div></div></div><div class="project-card project-item isotope-item js-id-Web-Article js-id-11 js-id-The-Space-Review"><div class=card><a href=/open-source-toolkit/post/is_open_sourcing_the_next_frontier_in_space_exploration_the_space_review/ class="card-image hover-overlay"><img src=/open-source-toolkit/post/is_open_sourcing_the_next_frontier_in_space_exploration_the_space_review/featured_hud109d9c66059217af2bab44795aa78ad_52218_550x0_resize_q90_lanczos_2.png alt class=img-responsive></a><div class=card-text><h4><a href=/open-source-toolkit/post/is_open_sourcing_the_next_frontier_in_space_exploration_the_space_review/>Is open sourcing the next frontier in space exploration?</a></h4><div class=article-style><p>Publishing date: &lsquo;2020-06-15&rsquo;
Published on: &lsquo;The Space Review&rsquo;
summary: &lsquo;Innovations in scientific research and data analysis can propel us forward in the field of space exploration. This article explores how Open Source practices can be used to boost the rate of innovation in the software and hardware involved in space exploration.</p></div></div></div></div><div class="project-card project-item isotope-item js-id-Research-Article js-id-None js-id-Hardware-X"><div class=card><a href=/open-source-toolkit/post/lowcost_and_precise_inline_pressure_sensor_housing_and_daq_for_use_in_laboratory_experiments_hardware_x/ class="card-image hover-overlay"><img src=/open-source-toolkit/post/lowcost_and_precise_inline_pressure_sensor_housing_and_daq_for_use_in_laboratory_experiments_hardware_x/featured_hud109d9c66059217af2bab44795aa78ad_52218_550x0_resize_q90_lanczos_2.png alt class=img-responsive></a><div class=card-text><h4><a href=/open-source-toolkit/post/lowcost_and_precise_inline_pressure_sensor_housing_and_daq_for_use_in_laboratory_experiments_hardware_x/>Low-cost and precise inline pressure sensor housing and DAQ for use in laboratory experiments</a></h4><div class=article-style><p>Publishing date: &lsquo;2020-06-15&rsquo;
Published on: &lsquo;Hardware X&rsquo;
summary: &lsquo;None&rsquo;
authors: &lsquo;Luke Goertzen, Nicole Mehr, Manuel Lopez, Chet Udell, John S.Selker&rsquo;
link to paper: &lsquo;https://doi.org/10.1016/j.ohx.2020.e00112'
Icons made by catkuro from www.flaticon.com</p></div></div></div></div><div class="project-card project-item isotope-item js-id-Research-Article js-id-None js-id-bioRxiv"><div class=card><a href=/open-source-toolkit/post/planktonscope_affordable_modular_imaging_platform_for_citizen_oceanography__biorxiv/ class="card-image hover-overlay"><img src=/open-source-toolkit/post/planktonscope_affordable_modular_imaging_platform_for_citizen_oceanography__biorxiv/featured_hud109d9c66059217af2bab44795aa78ad_52218_550x0_resize_q90_lanczos_2.png alt class=img-responsive></a><div class=card-text><h4><a href=/open-source-toolkit/post/planktonscope_affordable_modular_imaging_platform_for_citizen_oceanography__biorxiv/>PlanktonScope: Affordable modular imaging platform for citizen oceanography</a></h4><div class=article-style><p>Publishing date: &lsquo;2020-06-15&rsquo;
Published on: &lsquo;bioRxiv&rsquo;
summary: &lsquo;None&rsquo;
authors: &lsquo;Thibaut Pollina, Adam G. Larson, Fabien Lombard, Hongquan Li, Sebastien Colin, Colomban de Vargas, Manu Prakash&rsquo;
link to paper: &lsquo;https://doi.org/10.1101/2020.04.23.056978'
Icons made by catkuro from www.</p></div></div></div></div><div class="project-card project-item isotope-item js-id-Research-Article js-id-None js-id-bioRxiv"><div class=card><a href=/open-source-toolkit/post/sictec_an_inexpensive_easily_assembled_peltier_device_for_rapid_temperature_shifting_during_singlecell_imaging__biorxiv/ class="card-image hover-overlay"><img src=/open-source-toolkit/post/sictec_an_inexpensive_easily_assembled_peltier_device_for_rapid_temperature_shifting_during_singlecell_imaging__biorxiv/featured_hud109d9c66059217af2bab44795aa78ad_52218_550x0_resize_q90_lanczos_2.png alt class=img-responsive></a><div class=card-text><h4><a href=/open-source-toolkit/post/sictec_an_inexpensive_easily_assembled_peltier_device_for_rapid_temperature_shifting_during_singlecell_imaging__biorxiv/>SiCTeC: an inexpensive, easily assembled Peltier device for rapid temperature shifting during single-cell imaging</a></h4><div class=article-style><p>Publishing date: &lsquo;2020-06-15&rsquo;
Published on: &lsquo;bioRxiv&rsquo;
summary: &lsquo;None&rsquo;
authors: &lsquo;Benjamin D. Knapp, Lillian Zhu, Kerwyn Casey Huang&rsquo;
link to paper: &lsquo;https://doi.org/10.1101/2020.05.29.123166'
Icons made by catkuro from www.flaticon.com</p></div></div></div></div><div class="project-card project-item isotope-item js-id-Research-Article js-id-None js-id-bioRxiv"><div class=card><a href=/open-source-toolkit/post/the_incubot_a_3d_printerbased_microscope_for_longterm_live_cell_imaging_within_a_tissue_culture_incubator__biorxiv/ class="card-image hover-overlay"><img src=/open-source-toolkit/post/the_incubot_a_3d_printerbased_microscope_for_longterm_live_cell_imaging_within_a_tissue_culture_incubator__biorxiv/featured_hud109d9c66059217af2bab44795aa78ad_52218_550x0_resize_q90_lanczos_2.png alt class=img-responsive></a><div class=card-text><h4><a href=/open-source-toolkit/post/the_incubot_a_3d_printerbased_microscope_for_longterm_live_cell_imaging_within_a_tissue_culture_incubator__biorxiv/>The Incubot: A 3D Printer-Based Microscope for Long-Term Live Cell Imaging within a Tissue Culture Incubator</a></h4><div class=article-style><p>Publishing date: &lsquo;2020-06-15&rsquo;
Published on: &lsquo;bioRxiv&rsquo;
summary: &lsquo;Commercial live cell imaging systems represent a large financial burden to research groups, while current open source incubator microscopy systems lack adaptability and are sometimes inadequate for complex imaging experimentation.</p></div></div></div></div><div class="project-card project-item isotope-item js-id-Web-Article js-id-None js-id-Lab-On-The-Cheap"><div class=card><a href=/open-source-toolkit/post/the_science_cheep__1__lab_on_the_cheap/ class="card-image hover-overlay"><img src=/open-source-toolkit/post/the_science_cheep__1__lab_on_the_cheap/featured_hud109d9c66059217af2bab44795aa78ad_52218_550x0_resize_q90_lanczos_2.png alt class=img-responsive></a><div class=card-text><h4><a href=/open-source-toolkit/post/the_science_cheep__1__lab_on_the_cheap/>The Science Cheep – #1</a></h4><div class=article-style><p>Publishing date: &lsquo;2020-06-15&rsquo;
Published on: &lsquo;Lab On The Cheap&rsquo;
summary: &lsquo;The Science Cheep, written for Lab On The Cheap, provides you with updates in the DIY open-source world of science from the past week.</p></div></div></div></div><div class="project-card project-item isotope-item js-id-Web-Article js-id-None js-id-TechCrunch"><div class=card><a href=/open-source-toolkit/post/tinyml_is_giving_hardware_new_life__techcrunch/ class="card-image hover-overlay"><img src=/open-source-toolkit/post/tinyml_is_giving_hardware_new_life__techcrunch/featured_hud109d9c66059217af2bab44795aa78ad_52218_550x0_resize_q90_lanczos_2.png alt class=img-responsive></a><div class=card-text><h4><a href=/open-source-toolkit/post/tinyml_is_giving_hardware_new_life__techcrunch/>TinyML is giving hardware new life</a></h4><div class=article-style><p>Publishing date: &lsquo;2020-06-15&rsquo;
Published on: &lsquo;TechCrunch&rsquo;
summary: &lsquo;None&rsquo;
authors: &lsquo;Adam Benzion&rsquo;
link to paper: &lsquo;https://techcrunch.com/2020/05/29/tinyml-is-giving-hardware-new-life/'
Icons made by catkuro from www.flaticon.com</p></div></div></div></div><div class="project-card project-item isotope-item js-id-Research-Article js-id-None js-id-PLOS-ONE"><div class=card><a href=/open-source-toolkit/post/a_practical_3dprinted_soft_robotic_prosthetic_hand_with_multiarticulating_capabilities_plos_one/ class="card-image hover-overlay"><img src=/open-source-toolkit/post/a_practical_3dprinted_soft_robotic_prosthetic_hand_with_multiarticulating_capabilities_plos_one/featured_hud109d9c66059217af2bab44795aa78ad_52218_550x0_resize_q90_lanczos_2.png alt class=img-responsive></a><div class=card-text><h4><a href=/open-source-toolkit/post/a_practical_3dprinted_soft_robotic_prosthetic_hand_with_multiarticulating_capabilities_plos_one/>A practical 3D-printed soft robotic prosthetic hand with multi-articulating capabilities</a></h4><div class=article-style><p>Publishing date: &lsquo;2020-05-14&rsquo;
Published on: &lsquo;PLOS ONE&rsquo;
summary: &lsquo;None&rsquo;
authors: &lsquo;Alireza Mohammadi, Jim Lavranos, Hao Zhou, Rahim Mutlu, Gursel Alici, Ying Tan, Peter Choong, Denny Oetomo&rsquo;
link to paper: &lsquo;10.1371/journal.pone.0232766&rsquo;</p></div></div></div></div><div class="project-card project-item isotope-item js-id-Web-Article js-id-None js-id-The-Enterprisers-Project"><div class=card><a href=/open-source-toolkit/post/open_source_meeting_tools_3_things_to_know__the_enterprisers_project/ class="card-image hover-overlay"><img src=/open-source-toolkit/post/open_source_meeting_tools_3_things_to_know__the_enterprisers_project/featured_hud109d9c66059217af2bab44795aa78ad_52218_550x0_resize_q90_lanczos_2.png alt class=img-responsive></a><div class=card-text><h4><a href=/open-source-toolkit/post/open_source_meeting_tools_3_things_to_know__the_enterprisers_project/>Open source meeting tools: 3 things to know</a></h4><div class=article-style><p>Publishing date: &lsquo;2020-05-12&rsquo;
Published on: &lsquo;The Enterprisers Project&rsquo;
summary: &lsquo;Are you considering an open source alternative to standard videoconferencing software? Here, the author reviews some of the options such as Jitsi, Jami, and Signal.</p></div></div></div></div><div class="project-card project-item isotope-item js-id-Research-Article js-id-None js-id-PLOS-Computational-Biology"><div class=card><a href=/open-source-toolkit/post/proteinvr_webbased_molecular_visualization_in_virtual_reality_plos_computational_biology/ class="card-image hover-overlay"><img src=/open-source-toolkit/post/proteinvr_webbased_molecular_visualization_in_virtual_reality_plos_computational_biology/featured_hud109d9c66059217af2bab44795aa78ad_52218_550x0_resize_q90_lanczos_2.png alt class=img-responsive></a><div class=card-text><h4><a href=/open-source-toolkit/post/proteinvr_webbased_molecular_visualization_in_virtual_reality_plos_computational_biology/>ProteinVR: Web-based molecular visualization in virtual reality</a></h4><div class=article-style><p>Publishing date: &lsquo;2020-05-12&rsquo;
Published on: &lsquo;PLOS Computational Biology&rsquo;
summary: &lsquo;Channel Editor Summary: Have you ever wondered what proteins look like? In fact, accurately conceptualizing 3D protein structures is vital to both scientific research and education.</p></div></div></div></div><div class="project-card project-item isotope-item js-id-Web-Article js-id-None js-id-Nature"><div class=card><a href=/open-source-toolkit/post/synchronized_editing_the_future_of_collaborative_writing_nature/ class="card-image hover-overlay"><img src=/open-source-toolkit/post/synchronized_editing_the_future_of_collaborative_writing_nature/featured_hud109d9c66059217af2bab44795aa78ad_52218_550x0_resize_q90_lanczos_2.png alt class=img-responsive></a><div class=card-text><h4><a href=/open-source-toolkit/post/synchronized_editing_the_future_of_collaborative_writing_nature/>Synchronized editing: the future of collaborative writing</a></h4><div class=article-style><p>Publishing date: &lsquo;2020-05-12&rsquo;
Published on: &lsquo;Nature&rsquo;
summary: &lsquo;Draft scientific manuscripts tend to be confidential, however this article presents the approach of Olga Botvinnik who managed her paper like a piece of open-source software.</p></div></div></div></div><div class="project-card project-item isotope-item js-id-Web-Article js-id-None js-id-Information-Age"><div class=card><a href=/open-source-toolkit/post/tech_and_covid19_open_source_needed_for_contact_tracing_apps_information_age/ class="card-image hover-overlay"><img src=/open-source-toolkit/post/tech_and_covid19_open_source_needed_for_contact_tracing_apps_information_age/featured_hud109d9c66059217af2bab44795aa78ad_52218_550x0_resize_q90_lanczos_2.png alt class=img-responsive></a><div class=card-text><h4><a href=/open-source-toolkit/post/tech_and_covid19_open_source_needed_for_contact_tracing_apps_information_age/>Tech and Covid-19: open source needed for contact tracing apps</a></h4><div class=article-style><p>Publishing date: &lsquo;2020-05-12&rsquo;
Published on: &lsquo;Information Age&rsquo;
summary: &lsquo;The inevitable and necessary responses to COVID-19 from the lockdown itself, to the use of contact tracing apps raise concerns about our civil liberties that in a different time would all have been hotly debated over a considerable time period.</p></div></div></div></div><div class="project-card project-item isotope-item js-id-Commentary js-id-None js-id-PLOS-Biology"><div class=card><a href=/open-source-toolkit/post/leveraging_open_hardware_to_alleviate_the_burden_of_covid19_on_global_health_systems_plos_biology/ class="card-image hover-overlay"><img src=/open-source-toolkit/post/leveraging_open_hardware_to_alleviate_the_burden_of_covid19_on_global_health_systems_plos_biology/featured_hud109d9c66059217af2bab44795aa78ad_52218_550x0_resize_q90_lanczos_2.png alt class=img-responsive></a><div class=card-text><h4><a href=/open-source-toolkit/post/leveraging_open_hardware_to_alleviate_the_burden_of_covid19_on_global_health_systems_plos_biology/>Leveraging open hardware to alleviate the burden of COVID-19 on global health systems</a></h4><div class=article-style><p>Publishing date: &lsquo;2020-05-06&rsquo;
Published on: &lsquo;PLOS Biology&rsquo;
summary: &lsquo;With the current rapid spread of COVID-19, global health systems are increasingly overburdened by the sheer number of people that need diagnosis, isolation and treatment.</p></div></div></div></div><div class="project-card project-item isotope-item js-id-Research-Article js-id-None js-id-PLOS-Computational-Biology"><div class=card><a href=/open-source-toolkit/post/scedar_a_scalable_python_package_for_singlecell_rnaseq_exploratory_data_analysis_plos_computational_biology/ class="card-image hover-overlay"><img src=/open-source-toolkit/post/scedar_a_scalable_python_package_for_singlecell_rnaseq_exploratory_data_analysis_plos_computational_biology/featured_hud109d9c66059217af2bab44795aa78ad_52218_550x0_resize_q90_lanczos_2.png alt class=img-responsive></a><div class=card-text><h4><a href=/open-source-toolkit/post/scedar_a_scalable_python_package_for_singlecell_rnaseq_exploratory_data_analysis_plos_computational_biology/>Scedar: A scalable Python package for single-cell RNA-seq exploratory data analysis</a></h4><div class=article-style><p>Publishing date: &lsquo;2020-04-27&rsquo;
Published on: &lsquo;PLOS Computational Biology&rsquo;
summary: &lsquo;None&rsquo;
authors: &lsquo;Yuanchao Zhang, Man S. Kim, Erin R. Reichenberger, Ben Stear, Deanne M. Taylor&rsquo;
link to paper: &lsquo;10.1371/journal.pcbi.1007794&rsquo;
Icons made by catkuro from www.</p></div></div></div></div><div class="project-card project-item isotope-item js-id-Research-Article js-id-None js-id-PLOS-ONE"><div class=card><a href=/open-source-toolkit/post/dataflow_programming_for_the_analysis_of_molecular_dynamics_with_avis_an_analysis_and_visualization_software_application_plos_one/ class="card-image hover-overlay"><img src=/open-source-toolkit/post/dataflow_programming_for_the_analysis_of_molecular_dynamics_with_avis_an_analysis_and_visualization_software_application_plos_one/featured_hud109d9c66059217af2bab44795aa78ad_52218_550x0_resize_q90_lanczos_2.png alt class=img-responsive></a><div class=card-text><h4><a href=/open-source-toolkit/post/dataflow_programming_for_the_analysis_of_molecular_dynamics_with_avis_an_analysis_and_visualization_software_application_plos_one/>Dataflow programming for the analysis of molecular dynamics with AViS, an analysis and visualization software application</a></h4><div class=article-style><p>Publishing date: &lsquo;2020-04-21&rsquo;
Published on: &lsquo;PLOS ONE&rsquo;
summary: &lsquo;None&rsquo;
authors: &lsquo;Kai Pua, Daisuke Yuhara, Sho Ayuba, Kenji Yasuoka&rsquo;
link to paper: &lsquo;10.1371/journal.pone.0231714&rsquo;
Icons made by catkuro from www.flaticon.com</p></div></div></div></div><div class="project-card project-item isotope-item js-id-Research-Article js-id-None js-id-PLOS-Computational-Biology"><div class=card><a href=/open-source-toolkit/post/3d_mesh_processing_using_gamer_2_to_enable_reactiondiffusion_simulations_in_realistic_cellular_geometries_plos_computational_biology/ class="card-image hover-overlay"><img src=/open-source-toolkit/post/3d_mesh_processing_using_gamer_2_to_enable_reactiondiffusion_simulations_in_realistic_cellular_geometries_plos_computational_biology/featured_hud109d9c66059217af2bab44795aa78ad_52218_550x0_resize_q90_lanczos_2.png alt class=img-responsive></a><div class=card-text><h4><a href=/open-source-toolkit/post/3d_mesh_processing_using_gamer_2_to_enable_reactiondiffusion_simulations_in_realistic_cellular_geometries_plos_computational_biology/>3D mesh processing using GAMer 2 to enable reaction-diffusion simulations in realistic cellular geometries</a></h4><div class=article-style><p>Publishing date: &lsquo;2020-04-06&rsquo;
Published on: &lsquo;PLOS Computational Biology&rsquo;
summary: &lsquo;None&rsquo;
authors: &lsquo;Christopher T. Lee, Justin G. Laughlin, Nils Angliviel de La Beaumelle, Rommie E. Amaro, J. Andrew McCammon, Ravi Ramamoorthi, Michael Holst, Padmini Rangamani&rsquo;</p></div></div></div></div><div class="project-card project-item isotope-item js-id-Research-Article js-id-None js-id-Preprints.org"><div class=card><a href=/open-source-toolkit/post/leveraging_open_hardware_to_alleviate_the_burden_of_covid19_on_global_health_systems_preprintsorg/ class="card-image hover-overlay"><img src=/open-source-toolkit/post/leveraging_open_hardware_to_alleviate_the_burden_of_covid19_on_global_health_systems_preprintsorg/featured_hud109d9c66059217af2bab44795aa78ad_52218_550x0_resize_q90_lanczos_2.png alt class=img-responsive></a><div class=card-text><h4><a href=/open-source-toolkit/post/leveraging_open_hardware_to_alleviate_the_burden_of_covid19_on_global_health_systems_preprintsorg/>Leveraging open hardware to alleviate the burden of COVID-19 on global health systems</a></h4><div class=article-style><p>Publishing date: &lsquo;2020-04-06&rsquo;
Published on: &lsquo;Preprints.org&rsquo;
summary: &lsquo;In the global response to COVID-19 there is a shortfall of staffing, facilities, and equipment facing up to high demand for diagnosis, isolation, and treatment.</p></div></div></div></div><div class="project-card project-item isotope-item js-id-Web-Article js-id-None js-id-The-New-Stack"><div class=card><a href=/open-source-toolkit/post/open_source_hardware_the_rise_of_riscv__the_new_stack/ class="card-image hover-overlay"><img src=/open-source-toolkit/post/open_source_hardware_the_rise_of_riscv__the_new_stack/featured_hud109d9c66059217af2bab44795aa78ad_52218_550x0_resize_q90_lanczos_2.png alt class=img-responsive></a><div class=card-text><h4><a href=/open-source-toolkit/post/open_source_hardware_the_rise_of_riscv__the_new_stack/>Open Source Hardware: The Rise of RISC-V</a></h4><div class=article-style><p>Publishing date: &lsquo;2020-04-06&rsquo;
Published on: &lsquo;The New Stack&rsquo;
summary: &lsquo;This article looks at RISC-V. The RISC-V foundation provides a free and open source instruction set architecture for chips based on the long-established paradigms for reduced instruction set computing.</p></div></div></div></div><div class="project-card project-item isotope-item js-id-Web-Article js-id-None js-id-Hackaday"><div class=card><a href=/open-source-toolkit/post/opensource_neuroscience_hardware_hack_chat__hackaday/ class="card-image hover-overlay"><img src=/open-source-toolkit/post/opensource_neuroscience_hardware_hack_chat__hackaday/featured_hud109d9c66059217af2bab44795aa78ad_52218_550x0_resize_q90_lanczos_2.png alt class=img-responsive></a><div class=card-text><h4><a href=/open-source-toolkit/post/opensource_neuroscience_hardware_hack_chat__hackaday/>Open-Source Neuroscience Hardware Hack Chat</a></h4><div class=article-style><p>Publishing date: &lsquo;2020-04-06&rsquo;
Published on: &lsquo;Hackaday&rsquo;
summary: &lsquo;This article was written to promote the OPEN-SOURCE NEUROSCIENCE HARDWARE HACK CHAT that took place in February 2020, a transcript of which is available &lt;a href=https://hackaday.</p></div></div></div></div><div class="project-card project-item isotope-item js-id-Research-Article js-id-None js-id-Scientific-Reports"><div class=card><a href=/open-source-toolkit/post/opensource_3dprinted_peristaltic_pumps_for_small_volume_pointofcare_liquid_handling__scientific_reports/ class="card-image hover-overlay"><img src=/open-source-toolkit/post/opensource_3dprinted_peristaltic_pumps_for_small_volume_pointofcare_liquid_handling__scientific_reports/featured_hud109d9c66059217af2bab44795aa78ad_52218_550x0_resize_q90_lanczos_2.png alt class=img-responsive></a><div class=card-text><h4><a href=/open-source-toolkit/post/opensource_3dprinted_peristaltic_pumps_for_small_volume_pointofcare_liquid_handling__scientific_reports/>Open-source, 3D-printed Peristaltic Pumps for Small Volume Point-of-Care Liquid Handling</a></h4><div class=article-style><p>Publishing date: &lsquo;2020-04-06&rsquo;
Published on: &lsquo;Scientific Reports&rsquo;
summary: &lsquo;Microfluidic technologies are frequently employed as point-of-care diagnostic tools for improving time-to-diagnosis and improving patient outcomes in clinical settings. These microfluidic devices often are designed to operate with peripheral equipment for liquid handling that increases the cost and complexity of these systems and reduces their potential for widespread adoption in low resource healthcare applications.</p></div></div></div></div><div class="project-card project-item isotope-item js-id-Research-Article js-id-None js-id-Traffic"><div class=card><a href=/open-source-toolkit/post/superbeacons_opensource_probes_with_spontaneous_tuneable_blinking_compatible_with_livecell_superresolution_microscopy_traffic/ class="card-image hover-overlay"><img src=/open-source-toolkit/post/superbeacons_opensource_probes_with_spontaneous_tuneable_blinking_compatible_with_livecell_superresolution_microscopy_traffic/featured_hud109d9c66059217af2bab44795aa78ad_52218_550x0_resize_q90_lanczos_2.png alt class=img-responsive></a><div class=card-text><h4><a href=/open-source-toolkit/post/superbeacons_opensource_probes_with_spontaneous_tuneable_blinking_compatible_with_livecell_superresolution_microscopy_traffic/>Super‐beacons: Open‐source probes with spontaneous tuneable blinking compatible with live‐cell super‐resolution microscopy</a></h4><div class=article-style><p>Publishing date: &lsquo;2020-04-06&rsquo;
Published on: &lsquo;Traffic&rsquo;
summary: &lsquo;In this work Pereira et al present a new class of DNA‐based self‐quenching open‐source super‐resolution probes that they developed, named super beacons. These probes have photoswitching kinetics that can be tuned structurally, thermally and chemically.</p></div></div></div></div><div class="project-card project-item isotope-item js-id-Research-Article js-id-None js-id-Technological-Forecasting-and-Social-Change"><div class=card><a href=/open-source-toolkit/post/towards_national_policy_for_open_source_hardware_research_the_case_of_finland_technological_forecasting_and_social_change/ class="card-image hover-overlay"><img src=/open-source-toolkit/post/towards_national_policy_for_open_source_hardware_research_the_case_of_finland_technological_forecasting_and_social_change/featured_hud109d9c66059217af2bab44795aa78ad_52218_550x0_resize_q90_lanczos_2.png alt class=img-responsive></a><div class=card-text><h4><a href=/open-source-toolkit/post/towards_national_policy_for_open_source_hardware_research_the_case_of_finland_technological_forecasting_and_social_change/>Towards national policy for open source hardware research: The case of Finland</a></h4><div class=article-style><p>Publishing date: &lsquo;2020-04-06&rsquo;
Published on: &lsquo;Technological Forecasting and Social Change&rsquo;
summary: &lsquo;Free and open-source hardware (FOSH) is rapidly gaining momentum because it provides customized research hardware with over 90% savings compared to the costs of proprietary tools.</p></div></div></div></div><div class="project-card project-item isotope-item js-id-Research-Article js-id-None js-id-bioRxiv"><div class=card><a href=/open-source-toolkit/post/uc2__a_versatile_and_customizable_lowcost_3dprinted_optical_openstandard_for_microscopic_imaging__biorxiv/ class="card-image hover-overlay"><img src=/open-source-toolkit/post/uc2__a_versatile_and_customizable_lowcost_3dprinted_optical_openstandard_for_microscopic_imaging__biorxiv/featured_hud109d9c66059217af2bab44795aa78ad_52218_550x0_resize_q90_lanczos_2.png alt class=img-responsive></a><div class=card-text><h4><a href=/open-source-toolkit/post/uc2__a_versatile_and_customizable_lowcost_3dprinted_optical_openstandard_for_microscopic_imaging__biorxiv/>UC2 – A Versatile and Customizable low-cost 3D-printed Optical Open-Standard for microscopic imaging</a></h4><div class=article-style><p>Publishing date: &lsquo;2020-04-06&rsquo;
Published on: &lsquo;bioRxiv&rsquo;
summary: &lsquo;Channel Editor Summary: UC2 is an inexpensive 3D-printed microscopy toolbox that takes modular development seriously, they are a great example of a project that has thought through assembly standards, extensibility and operability from the start of their project and have some impressive assemblies capable of hologrpahic, confocal and light sheet microscopy.</p></div></div></div></div><div class="project-card project-item isotope-item js-id-Research-Article js-id-None js-id-bioRxiv"><div class=card><a href=/open-source-toolkit/post/biteoscope_an_open_platform_to_study_mosquito_bloodfeeding_behavior__biorxiv/ class="card-image hover-overlay"><img src=/open-source-toolkit/post/biteoscope_an_open_platform_to_study_mosquito_bloodfeeding_behavior__biorxiv/featured_hud109d9c66059217af2bab44795aa78ad_52218_550x0_resize_q90_lanczos_2.png alt class=img-responsive></a><div class=card-text><h4><a href=/open-source-toolkit/post/biteoscope_an_open_platform_to_study_mosquito_bloodfeeding_behavior__biorxiv/>BiteOscope: an open platform to study mosquito blood-feeding behavior</a></h4><div class=article-style><p>Publishing date: &lsquo;2020-04-03&rsquo;
Published on: &lsquo;bioRxiv&rsquo;
summary: &lsquo;None&rsquo;
authors: &lsquo;Felix JH Hol, Louis Lambrechts, Manu Prakash&rsquo;
link to paper: &lsquo;https://doi.org/10.1101/2020.02.19.955641'
Icons made by catkuro from www.flaticon.com</p></div></div></div></div><div class="project-card project-item isotope-item js-id-Research-Article js-id-None js-id-bioRxiv"><div class=card><a href=/open-source-toolkit/post/led_zappelin_an_open_source_led_controller_for_arbitrary_spectrum_visual_stimulation_and_optogenetics_during_2photon_imaging__biorxiv_biorxiv/ class="card-image hover-overlay"><img src=/open-source-toolkit/post/led_zappelin_an_open_source_led_controller_for_arbitrary_spectrum_visual_stimulation_and_optogenetics_during_2photon_imaging__biorxiv_biorxiv/featured_hud109d9c66059217af2bab44795aa78ad_52218_550x0_resize_q90_lanczos_2.png alt class=img-responsive></a><div class=card-text><h4><a href=/open-source-toolkit/post/led_zappelin_an_open_source_led_controller_for_arbitrary_spectrum_visual_stimulation_and_optogenetics_during_2photon_imaging__biorxiv_biorxiv/>LED Zappelin’: An open source LED controller for arbitrary spectrum visual stimulation and optogenetics during 2-photon imaging | bioRxiv</a></h4><div class=article-style><p>Publishing date: &lsquo;2020-04-03&rsquo;
Published on: &lsquo;bioRxiv&rsquo;
summary: &lsquo;None&rsquo;
authors: &lsquo;M.J.Y. Zimmermann, A.M. Chagas, P. Bartel, S. Pop, L.L. Prieto Godino, T. Baden&rsquo;
link to paper: &lsquo;https://doi.org/10.1101/2020.03.10.986000'
Icons made by catkuro from www.</p></div></div></div></div><div class="project-card project-item isotope-item js-id-Research-Article js-id-None js-id-Hardware-X-"><div class=card><a href=/open-source-toolkit/post/microusv_a_lowcost_platform_for_indoor_marine_swarm_robotics_research_hardware_x_/ class="card-image hover-overlay"><img src=/open-source-toolkit/post/microusv_a_lowcost_platform_for_indoor_marine_swarm_robotics_research_hardware_x_/featured_hud109d9c66059217af2bab44795aa78ad_52218_550x0_resize_q90_lanczos_2.png alt class=img-responsive></a><div class=card-text><h4><a href=/open-source-toolkit/post/microusv_a_lowcost_platform_for_indoor_marine_swarm_robotics_research_hardware_x_/>microUSV: A low-cost platform for indoor marine swarm robotics research</a></h4><div class=article-style><p>Publishing date: &lsquo;2020-04-03&rsquo;
Published on: &lsquo;Hardware X '
summary: &lsquo;None&rsquo;
authors: &lsquo;Calvin Gregory, Andrew Vardy&rsquo;
link to paper: &lsquo;https://doi.org/10.1016/j.ohx.2020.e00105'
Icons made by catkuro from www.flaticon.com</p></div></div></div></div><div class="project-card project-item isotope-item js-id-Research-Article js-id-None js-id-Hardware-X"><div class=card><a href=/open-source-toolkit/post/opensource_five_degree_of_freedom_motion_platform_for_investigating_fishrobot_interaction_hardware_x/ class="card-image hover-overlay"><img src=/open-source-toolkit/post/opensource_five_degree_of_freedom_motion_platform_for_investigating_fishrobot_interaction_hardware_x/featured_hud109d9c66059217af2bab44795aa78ad_52218_550x0_resize_q90_lanczos_2.png alt class=img-responsive></a><div class=card-text><h4><a href=/open-source-toolkit/post/opensource_five_degree_of_freedom_motion_platform_for_investigating_fishrobot_interaction_hardware_x/>Open-source five degree of freedom motion platform for investigating fish-robot interaction</a></h4><div class=article-style><p>Publishing date: &lsquo;2020-04-03&rsquo;
Published on: &lsquo;Hardware X&rsquo;
summary: &lsquo;None&rsquo;
authors: &lsquo;Brent Utter, Alexander Brown&rsquo;
link to paper: &lsquo;https://doi.org/10.1016/j.ohx.2020.e00107'
Icons made by catkuro from www.flaticon.com</p></div></div></div></div><div class="project-card project-item isotope-item js-id-Research-Article js-id-None js-id-Hardware-X"><div class=card><a href=/open-source-toolkit/post/plaspi_marine_cameras_opensource_affordable_camera_systems_for_time_series_marine_studies_hardware_x/ class="card-image hover-overlay"><img src=/open-source-toolkit/post/plaspi_marine_cameras_opensource_affordable_camera_systems_for_time_series_marine_studies_hardware_x/featured_hud109d9c66059217af2bab44795aa78ad_52218_550x0_resize_q90_lanczos_2.png alt class=img-responsive></a><div class=card-text><h4><a href=/open-source-toolkit/post/plaspi_marine_cameras_opensource_affordable_camera_systems_for_time_series_marine_studies_hardware_x/>PlasPI marine cameras: Open-source, affordable camera systems for time series marine studies</a></h4><div class=article-style><p>Publishing date: &lsquo;2020-04-03&rsquo;
Published on: &lsquo;Hardware X&rsquo;
summary: &lsquo;None&rsquo;
authors: &lsquo;Autun Purser, Ulrich Hoge, Johannes Lemburg, Yasemin Bodur, Elena Schiller, Janine Ludszuweit, Jens Greinert, Frank Wenzhöfer&rsquo;
link to paper: &lsquo;https://doi.org/10.1016/j.ohx.2020.e00102'</p></div></div></div></div><div class="project-card project-item isotope-item js-id-Research-Article js-id-None js-id-bioRxiv"><div class=card><a href=/open-source-toolkit/post/rpg_a_lowcost_opensource_highperformance_solution_for_displaying_visual_stimuli__biorxiv/ class="card-image hover-overlay"><img src=/open-source-toolkit/post/rpg_a_lowcost_opensource_highperformance_solution_for_displaying_visual_stimuli__biorxiv/featured_hud109d9c66059217af2bab44795aa78ad_52218_550x0_resize_q90_lanczos_2.png alt class=img-responsive></a><div class=card-text><h4><a href=/open-source-toolkit/post/rpg_a_lowcost_opensource_highperformance_solution_for_displaying_visual_stimuli__biorxiv/>RPG: A low-cost, open-source, high-performance solution for displaying visual stimuli</a></h4><div class=article-style><p>Publishing date: &lsquo;2020-04-03&rsquo;
Published on: &lsquo;bioRxiv&rsquo;
summary: &lsquo;None&rsquo;
authors: &lsquo;Vivian Imbriotis, Adam Ranson, William M Connelly&rsquo;
link to paper: &lsquo;https://doi.org/10.1101/2020.03.05.979724'
Icons made by catkuro from www.flaticon.com</p></div></div></div></div><div class="project-card project-item isotope-item js-id-Research-Article js-id-None js-id-bioRxiv"><div class=card><a href=/open-source-toolkit/post/safe_acoustics_an_opensource_realtime_ecoacoustic_monitoring_network_in_the_tropical_rainforests_of_borneo__biorxiv/ class="card-image hover-overlay"><img src=/open-source-toolkit/post/safe_acoustics_an_opensource_realtime_ecoacoustic_monitoring_network_in_the_tropical_rainforests_of_borneo__biorxiv/featured_hud109d9c66059217af2bab44795aa78ad_52218_550x0_resize_q90_lanczos_2.png alt class=img-responsive></a><div class=card-text><h4><a href=/open-source-toolkit/post/safe_acoustics_an_opensource_realtime_ecoacoustic_monitoring_network_in_the_tropical_rainforests_of_borneo__biorxiv/>SAFE Acoustics: an open-source, real-time eco-acoustic monitoring network in the tropical rainforests of Borneo</a></h4><div class=article-style><p>Publishing date: &lsquo;2020-04-03&rsquo;
Published on: &lsquo;bioRxiv&rsquo;
summary: &lsquo;None&rsquo;
authors: &lsquo;Sarab S. Sethi, Robert M. Ewers, Nick S. Jones, Aaron Signorelli, Lorenzo Picinali, C. David L. Orme&rsquo;
link to paper: &lsquo;https://doi.org/10.1101/2020.02.27.968867'</p></div></div></div></div><div class="project-card project-item isotope-item js-id-Commentary js-id-None js-id-PLOS-Computational-Biology"><div class=card><a href=/open-source-toolkit/post/ten_simple_rules_for_providing_effective_bioinformatics_research_support_plos_computational_biology/ class="card-image hover-overlay"><img src=/open-source-toolkit/post/ten_simple_rules_for_providing_effective_bioinformatics_research_support_plos_computational_biology/featured_hud109d9c66059217af2bab44795aa78ad_52218_550x0_resize_q90_lanczos_2.png alt class=img-responsive></a><div class=card-text><h4><a href=/open-source-toolkit/post/ten_simple_rules_for_providing_effective_bioinformatics_research_support_plos_computational_biology/>Ten simple rules for providing effective bioinformatics research support</a></h4><div class=article-style><p>Publishing date: &lsquo;2020-03-26&rsquo;
Published on: &lsquo;PLOS Computational Biology&rsquo;
summary: &lsquo;None&rsquo;
authors: &lsquo;Judit Kumuthini, Michael Chimenti, Sven Nahnsen, Alexander Peltzer, Rebone Meraba, Ross McFadyen, Gordon Wells, Deanne Taylor, Mark Maienschein-Cline, Jian-Liang Li, Jyothi Thimmapuram, Radha Murthy-Karuturi, Lyndon Zass&rsquo;</p></div></div></div></div><div class="project-card project-item isotope-item js-id-Research-Article js-id-None js-id-PLOS-ONE"><div class=card><a href=/open-source-toolkit/post/multiple_social_platforms_reveal_actionable_signals_for_software_vulnerability_awareness_a_study_of_github_twitter_and_reddit_plos_one/ class="card-image hover-overlay"><img src=/open-source-toolkit/post/multiple_social_platforms_reveal_actionable_signals_for_software_vulnerability_awareness_a_study_of_github_twitter_and_reddit_plos_one/featured_hud109d9c66059217af2bab44795aa78ad_52218_550x0_resize_q90_lanczos_2.png alt class=img-responsive></a><div class=card-text><h4><a href=/open-source-toolkit/post/multiple_social_platforms_reveal_actionable_signals_for_software_vulnerability_awareness_a_study_of_github_twitter_and_reddit_plos_one/>Multiple social platforms reveal actionable signals for software vulnerability awareness: A study of GitHub, Twitter and Reddit</a></h4><div class=article-style><p>Publishing date: &lsquo;2020-03-24&rsquo;
Published on: &lsquo;PLOS ONE&rsquo;
summary: &lsquo;None&rsquo;
authors: &lsquo;Prasha Shrestha, Arun Sathanur, Suraj Maharjan, Emily Saldanha, Dustin Arendt, Svitlana Volkova&rsquo;
link to paper: &lsquo;10.1371/journal.pone.0230250&rsquo;
Icons made by catkuro from www.</p></div></div></div></div><div class="project-card project-item isotope-item js-id-Research-Article js-id-None js-id-PLOS-ONE"><div class=card><a href=/open-source-toolkit/post/a_portable_lowcost_device_for_precise_control_of_specimen_temperature_under_stereomicroscopes_plos_one/ class="card-image hover-overlay"><img src=/open-source-toolkit/post/a_portable_lowcost_device_for_precise_control_of_specimen_temperature_under_stereomicroscopes_plos_one/featured_hud109d9c66059217af2bab44795aa78ad_52218_550x0_resize_q90_lanczos_2.png alt class=img-responsive></a><div class=card-text><h4><a href=/open-source-toolkit/post/a_portable_lowcost_device_for_precise_control_of_specimen_temperature_under_stereomicroscopes_plos_one/>A portable, low-cost device for precise control of specimen temperature under stereomicroscopes</a></h4><div class=article-style><p>Publishing date: &lsquo;2020-03-11&rsquo;
Published on: &lsquo;PLOS ONE&rsquo;
summary: &lsquo;None&rsquo;
authors: &lsquo;Nicholas D. Testa, Samiksha Kaul, Kim N. Le, Mei Zhan, Hang Lu, Annalise B. Paaby&rsquo;
link to paper: &lsquo;10.1371/journal.pone.0230241&rsquo;
Icons made by catkuro from www.</p></div></div></div></div><div class="project-card project-item isotope-item js-id-Research-Article"><div class=card><a href=/open-source-toolkit/post/bornagain_software_for_simulating_and_fitting_grazingincidence_smallangle_scattering_journal-of-applied-crystallography/ class="card-image hover-overlay"><img src=/open-source-toolkit/post/bornagain_software_for_simulating_and_fitting_grazingincidence_smallangle_scattering_journal-of-applied-crystallography/featured_hud109d9c66059217af2bab44795aa78ad_52218_550x0_resize_q90_lanczos_2.png alt class=img-responsive></a><div class=card-text><h4><a href=/open-source-toolkit/post/bornagain_software_for_simulating_and_fitting_grazingincidence_smallangle_scattering_journal-of-applied-crystallography/>BornAgain: software for simulating and fitting grazing-incidence small-angle scattering</a></h4><div class=article-style><p>Publishing date: &lsquo;2020-03-03&rsquo;
Published on: &lsquo;Journal of Applied Crystallography&rsquo;
summary: &lsquo;Journal of Applied Crystallography&rsquo;
authors: &lsquo;G. Pospelov, W. Van Herck, J. Burle, J. M. Carmona Loaiza, C. Durniak, J. M. Fisher, M.</p></div></div></div></div><div class="project-card project-item isotope-item js-id-Research-Article js-id-None js-id-bioRxiv"><div class=card><a href=/open-source-toolkit/post/compressive_big_data_analytics_an_ensemble_metaalgorithm_for_highdimensional_multisource_datasets_biorxiv/ class="card-image hover-overlay"><img src=/open-source-toolkit/post/compressive_big_data_analytics_an_ensemble_metaalgorithm_for_highdimensional_multisource_datasets_biorxiv/featured_hud109d9c66059217af2bab44795aa78ad_52218_550x0_resize_q90_lanczos_2.png alt class=img-responsive></a><div class=card-text><h4><a href=/open-source-toolkit/post/compressive_big_data_analytics_an_ensemble_metaalgorithm_for_highdimensional_multisource_datasets_biorxiv/>Compressive Big Data Analytics: An Ensemble Meta-Algorithm for High-dimensional Multisource Datasets</a></h4><div class=article-style><p>Publishing date: &lsquo;2020-03-03&rsquo;
Published on: &lsquo;bioRxiv&rsquo;
summary: &lsquo;None&rsquo;
authors: &lsquo;Simeone Marino, Yi Zhao, Nina Zhou, Yiwang Zhou, Arthur Toga, Lu Zhao, Yingsi Jian, Yichen Yang, Yehu Chen, Qiucheng Wu, Jessica Wild, Brandon Cummings, Ivo D.</p></div></div></div></div><div class="project-card project-item isotope-item js-id-Research-Article js-id-None js-id-GigaScience"><div class=card><a href=/open-source-toolkit/post/hamap_as_sparql_rulesa_portable_annotation_pipeline_for_genomes_and_proteomes__gigascience/ class="card-image hover-overlay"><img src=/open-source-toolkit/post/hamap_as_sparql_rulesa_portable_annotation_pipeline_for_genomes_and_proteomes__gigascience/featured_hud109d9c66059217af2bab44795aa78ad_52218_550x0_resize_q90_lanczos_2.png alt class=img-responsive></a><div class=card-text><h4><a href=/open-source-toolkit/post/hamap_as_sparql_rulesa_portable_annotation_pipeline_for_genomes_and_proteomes__gigascience/>HAMAP as SPARQL rules—A portable annotation pipeline for genomes and proteomes</a></h4><div class=article-style><p>Publishing date: &lsquo;2020-03-03&rsquo;
Published on: &lsquo;GigaScience&rsquo;
summary: &lsquo;Genome and proteome annotation pipelines are generally custom built and not easily reusable by other groups. This leads to duplication of effort, increased costs, and suboptimal annotation quality.</p></div></div></div></div><div class="project-card project-item isotope-item js-id-Research-Article js-id-None js-id-PLOS-ONE"><div class=card><a href=/open-source-toolkit/post/metrics_for_graph_comparison_a_practitioners_guide_plos_one/ class="card-image hover-overlay"><img src=/open-source-toolkit/post/metrics_for_graph_comparison_a_practitioners_guide_plos_one/featured_hud109d9c66059217af2bab44795aa78ad_52218_550x0_resize_q90_lanczos_2.png alt class=img-responsive></a><div class=card-text><h4><a href=/open-source-toolkit/post/metrics_for_graph_comparison_a_practitioners_guide_plos_one/>Metrics for graph comparison: A practitioner’s guide</a></h4><div class=article-style><p>Publishing date: &lsquo;2020-03-03&rsquo;
Published on: &lsquo;PLOS ONE&rsquo;
summary: &lsquo;Comparison of graph structure is a ubiquitous task in data analysis and machine learning, with diverse applications in fields such as neuroscience, cyber security, social network analysis, and bioinformatics.</p></div></div></div></div><div class="project-card project-item isotope-item js-id-Research-Article js-id-None js-id-BMC-Bioinformatics"><div class=card><a href=/open-source-toolkit/post/multitemplate_matching_a_versatile_tool_for_objectlocalization_in_microscopy_images___bmc_bioinformatics/ class="card-image hover-overlay"><img src=/open-source-toolkit/post/multitemplate_matching_a_versatile_tool_for_objectlocalization_in_microscopy_images___bmc_bioinformatics/featured_hud109d9c66059217af2bab44795aa78ad_52218_550x0_resize_q90_lanczos_2.png alt class=img-responsive></a><div class=card-text><h4><a href=/open-source-toolkit/post/multitemplate_matching_a_versatile_tool_for_objectlocalization_in_microscopy_images___bmc_bioinformatics/>Multi-template matching: a versatile tool for object-localization in microscopy images</a></h4><div class=article-style><p>Publishing date: &lsquo;2020-03-03&rsquo;
Published on: &lsquo;BMC Bioinformatics&rsquo;
summary: &lsquo;The localization of objects of interest is a key initial step in most image analysis workflows. For biomedical image data, classical image-segmentation methods like thresholding or edge detection are typically used.</p></div></div></div></div><div class="project-card project-item isotope-item js-id-Research-Article js-id-None js-id-BMC-Bioinformatics-"><div class=card><a href=/open-source-toolkit/post/panev_an_r_package_for_a_pathwaybased_network_visualization__bmc_bioinformatics_/ class="card-image hover-overlay"><img src=/open-source-toolkit/post/panev_an_r_package_for_a_pathwaybased_network_visualization__bmc_bioinformatics_/featured_hud109d9c66059217af2bab44795aa78ad_52218_550x0_resize_q90_lanczos_2.png alt class=img-responsive></a><div class=card-text><h4><a href=/open-source-toolkit/post/panev_an_r_package_for_a_pathwaybased_network_visualization__bmc_bioinformatics_/>PANEV: an R package for a pathway-based network visualization</a></h4><div class=article-style><p>Publishing date: &lsquo;2020-03-03&rsquo;
Published on: &lsquo;BMC Bioinformatics '
summary: &lsquo;None&rsquo;
authors: &lsquo;Valentino Palombo, Marco Milanesi, Gabriella Sferra, Stefano Capomaccio, Sandy Sgorlon & Mariasilvia D’Andrea '
link to paper: &lsquo;https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-020-3371-7'
Icons made by catkuro from www.</p></div></div></div></div><div class="project-card project-item isotope-item js-id-Research-Article js-id-None js-id-PLOS-ONE"><div class=card><a href=/open-source-toolkit/post/pykneer_an_image_analysis_workflow_for_open_and_reproducible_research_on_femoral_knee_cartilage_plos_one/ class="card-image hover-overlay"><img src=/open-source-toolkit/post/pykneer_an_image_analysis_workflow_for_open_and_reproducible_research_on_femoral_knee_cartilage_plos_one/featured_hud109d9c66059217af2bab44795aa78ad_52218_550x0_resize_q90_lanczos_2.png alt class=img-responsive></a><div class=card-text><h4><a href=/open-source-toolkit/post/pykneer_an_image_analysis_workflow_for_open_and_reproducible_research_on_femoral_knee_cartilage_plos_one/>pyKNEEr: An image analysis workflow for open and reproducible research on femoral knee cartilage</a></h4><div class=article-style><p>Publishing date: &lsquo;2020-03-03&rsquo;
Published on: &lsquo;PLOS ONE&rsquo;
summary: &lsquo;Channel Editor Summary: Sometimes, all the available algorithms to perform a scientific task are either closed source, or just too complex to easily be re-used by others.</p></div></div></div></div><div class="project-card project-item isotope-item js-id-Research-Article js-id-None js-id-PLOS-ONE"><div class=card><a href=/open-source-toolkit/post/repositorybased_plasmid_design_plos_one/ class="card-image hover-overlay"><img src=/open-source-toolkit/post/repositorybased_plasmid_design_plos_one/featured_hud109d9c66059217af2bab44795aa78ad_52218_550x0_resize_q90_lanczos_2.png alt class=img-responsive></a><div class=card-text><h4><a href=/open-source-toolkit/post/repositorybased_plasmid_design_plos_one/>Repository-based plasmid design</a></h4><div class=article-style><p>Publishing date: &lsquo;2020-03-03&rsquo;
Published on: &lsquo;PLOS ONE&rsquo;
summary: &lsquo;There was an explosion in the amount of commercially available DNA in sequence repositories over the last decade. The number of such plasmids increased from 12,000 to over 300,000 among three of the largest repositories: iGEM, Addgene, and DNASU.</p></div></div></div></div><div class="project-card project-item isotope-item js-id-Research-Article js-id-None js-id-PLOS-Computational-Biology"><div class=card><a href=/open-source-toolkit/post/xpressyourself_enhancing_standardizing_and_automating_ribosome_profiling_computational_analyses_yields_improved_insight_into_data_plos_computational_biology/ class="card-image hover-overlay"><img src=/open-source-toolkit/post/xpressyourself_enhancing_standardizing_and_automating_ribosome_profiling_computational_analyses_yields_improved_insight_into_data_plos_computational_biology/featured_hud109d9c66059217af2bab44795aa78ad_52218_550x0_resize_q90_lanczos_2.png alt class=img-responsive></a><div class=card-text><h4><a href=/open-source-toolkit/post/xpressyourself_enhancing_standardizing_and_automating_ribosome_profiling_computational_analyses_yields_improved_insight_into_data_plos_computational_biology/>XPRESSyourself: Enhancing, standardizing, and automating ribosome profiling computational analyses yields improved insight into data</a></h4><div class=article-style><p>Publishing date: &lsquo;2020-03-03&rsquo;
Published on: &lsquo;PLOS Computational Biology&rsquo;
summary: &lsquo;Ribosome profiling, an application of nucleic acid sequencing for monitoring ribosome activity, has revolutionized our understanding of protein translation dynamics. This technique has been available for a decade, yet the current state and standardization of publicly available computational tools for these data is bleak.</p></div></div></div></div><div class="project-card project-item isotope-item js-id-Research-Article js-id-None js-id-PLOS-Computational-Biology"><div class=card><a href=/open-source-toolkit/post/rainbow_haplotypebased_genomewide_association_study_using_a_novel_snpset_method_plos_computational_biology/ class="card-image hover-overlay"><img src=/open-source-toolkit/post/rainbow_haplotypebased_genomewide_association_study_using_a_novel_snpset_method_plos_computational_biology/featured_hud109d9c66059217af2bab44795aa78ad_52218_550x0_resize_q90_lanczos_2.png alt class=img-responsive></a><div class=card-text><h4><a href=/open-source-toolkit/post/rainbow_haplotypebased_genomewide_association_study_using_a_novel_snpset_method_plos_computational_biology/>RAINBOW: Haplotype-based genome-wide association study using a novel SNP-set method</a></h4><div class=article-style><p>Publishing date: &lsquo;2020-02-14&rsquo;
Published on: &lsquo;PLOS Computational Biology&rsquo;
summary: &lsquo;None&rsquo;
authors: &lsquo;Kosuke Hamazaki, Hiroyoshi Iwata&rsquo;
link to paper: &lsquo;10.1371/journal.pcbi.1007663&rsquo;
Icons made by catkuro from www.flaticon.com</p></div></div></div></div><div class="project-card project-item isotope-item js-id-Research-Article js-id-None js-id-eNeuro"><div class=card><a href=/open-source-toolkit/post/an_opensource_automated_homecage_sipper_device_for_monitoring_liquid_ingestive_behavior_in_rodents__eneuro/ class="card-image hover-overlay"><img src=/open-source-toolkit/post/an_opensource_automated_homecage_sipper_device_for_monitoring_liquid_ingestive_behavior_in_rodents__eneuro/featured_hud109d9c66059217af2bab44795aa78ad_52218_550x0_resize_q90_lanczos_2.png alt class=img-responsive></a><div class=card-text><h4><a href=/open-source-toolkit/post/an_opensource_automated_homecage_sipper_device_for_monitoring_liquid_ingestive_behavior_in_rodents__eneuro/>An Open-Source, Automated Home-Cage Sipper Device for Monitoring Liquid Ingestive Behavior in Rodents</a></h4><div class=article-style><p>Publishing date: &lsquo;2020-02-07&rsquo;
Published on: &lsquo;eNeuro&rsquo;
summary: &lsquo;None&rsquo;
authors: &lsquo;Elizabeth Godynyuk, Maya N. Bluitt, Jessica R. Tooley, Alexxai V. Kravitz and Meaghan C. Creed&rsquo;
link to paper: &lsquo;https://www.eneuro.org/content/6/5/ENEURO.0292-19.2019'
Icons made by catkuro from www.</p></div></div></div></div><div class="project-card project-item isotope-item js-id-Commentary js-id-None js-id-PLOS-Biology"><div class=card><a href=/open-source-toolkit/post/electropen_an_ultralowcost_electricityfree_portable_electroporator_plos_biology/ class="card-image hover-overlay"><img src=/open-source-toolkit/post/electropen_an_ultralowcost_electricityfree_portable_electroporator_plos_biology/featured_hud109d9c66059217af2bab44795aa78ad_52218_550x0_resize_q90_lanczos_2.png alt class=img-responsive></a><div class=card-text><h4><a href=/open-source-toolkit/post/electropen_an_ultralowcost_electricityfree_portable_electroporator_plos_biology/>ElectroPen: An ultra-low–cost, electricity-free, portable electroporator</a></h4><div class=article-style><p>Publishing date: &lsquo;2020-02-07&rsquo;
Published on: &lsquo;PLOS Biology&rsquo;
summary: &lsquo;None&rsquo;
authors: &lsquo;Gaurav Byagathvalli, Soham Sinha, Yan Zhang, Mark P. Styczynski, Janet Standeven, M. Saad Bhamla&rsquo;
link to paper: &lsquo;10.1371/journal.pbio.3000589&rsquo;
Icons made by catkuro from www.</p></div></div></div></div><div class="project-card project-item isotope-item js-id-Research-Article js-id-None js-id-PLOS-ONE"><div class=card><a href=/open-source-toolkit/post/exploiting_open_source_3d_printer_architecture_for_laboratory_robotics_to_automate_highthroughput_timelapse_imaging_for_analytical_microbiology_plos_one/ class="card-image hover-overlay"><img src=/open-source-toolkit/post/exploiting_open_source_3d_printer_architecture_for_laboratory_robotics_to_automate_highthroughput_timelapse_imaging_for_analytical_microbiology_plos_one/featured_hud109d9c66059217af2bab44795aa78ad_52218_550x0_resize_q90_lanczos_2.png alt class=img-responsive></a><div class=card-text><h4><a href=/open-source-toolkit/post/exploiting_open_source_3d_printer_architecture_for_laboratory_robotics_to_automate_highthroughput_timelapse_imaging_for_analytical_microbiology_plos_one/>Exploiting open source 3D printer architecture for laboratory robotics to automate high-throughput time-lapse imaging for analytical microbiology</a></h4><div class=article-style><p>Publishing date: &lsquo;2020-02-07&rsquo;
Published on: &lsquo;PLOS ONE&rsquo;
summary: &lsquo;None&rsquo;
authors: &lsquo;Sarah H. Needs, Tai The Diep, Stephanie P. Bull, Anton Lindley-Decaire, Partha Ray, Alexander D. Edwards&rsquo;
link to paper: &lsquo;10.1371/journal.pone.0224878&rsquo;</p></div></div></div></div><div class="project-card project-item isotope-item js-id-Research-Article js-id-None js-id-PLOS-ONE"><div class=card><a href=/open-source-toolkit/post/oneshot_phaserecovery_using_a_cellphone_rgb_camera_on_a_jaminlebedeff_microscope_plos_one/ class="card-image hover-overlay"><img src=/open-source-toolkit/post/oneshot_phaserecovery_using_a_cellphone_rgb_camera_on_a_jaminlebedeff_microscope_plos_one/featured_hud109d9c66059217af2bab44795aa78ad_52218_550x0_resize_q90_lanczos_2.png alt class=img-responsive></a><div class=card-text><h4><a href=/open-source-toolkit/post/oneshot_phaserecovery_using_a_cellphone_rgb_camera_on_a_jaminlebedeff_microscope_plos_one/>One-shot phase-recovery using a cellphone RGB camera on a Jamin-Lebedeff microscope</a></h4><div class=article-style><p>Publishing date: &lsquo;2020-02-07&rsquo;
Published on: &lsquo;PLOS ONE&rsquo;
summary: &lsquo;None&rsquo;
authors: &lsquo;Benedict Diederich, Barbora Marsikova, Brad Amos, Rainer Heintzmann&rsquo;
link to paper: &lsquo;10.1371/journal.pone.0227096&rsquo;
Icons made by catkuro from www.flaticon.com</p></div></div></div></div><div class="project-card project-item isotope-item js-id-Research-Article js-id-None js-id-bioRxiv"><div class=card><a href=/open-source-toolkit/post/opensource_3d_printed_airjet_for_generating_monodispersed_alginate_microhydrogels__biorxiv/ class="card-image hover-overlay"><img src=/open-source-toolkit/post/opensource_3d_printed_airjet_for_generating_monodispersed_alginate_microhydrogels__biorxiv/featured_hud109d9c66059217af2bab44795aa78ad_52218_550x0_resize_q90_lanczos_2.png alt class=img-responsive></a><div class=card-text><h4><a href=/open-source-toolkit/post/opensource_3d_printed_airjet_for_generating_monodispersed_alginate_microhydrogels__biorxiv/>Open-source 3D printed air-jet for generating monodispersed alginate microhydrogels</a></h4><div class=article-style><p>Publishing date: &lsquo;2020-02-07&rsquo;
Published on: &lsquo;bioRxiv&rsquo;
summary: &lsquo;None&rsquo;
authors: &lsquo;Dustin J. Hadley, Kevin T. Campbell, Marina H. Gabriel, Eduardo A. Silva&rsquo;
link to paper: &lsquo;https://www.biorxiv.org/content/10.1101/804849v2'
Icons made by catkuro from www.</p></div></div></div></div><div class="project-card project-item isotope-item js-id-Research-Article js-id-None js-id-J.-Chem.-Educ"><div class=card><a href=/open-source-toolkit/post/sweepstat_a_buildityourself_twoelectrode_potentiostat_for_macroelectrode_and_ultramicroelectrode_studies_j_chem_educ/ class="card-image hover-overlay"><img src=/open-source-toolkit/post/sweepstat_a_buildityourself_twoelectrode_potentiostat_for_macroelectrode_and_ultramicroelectrode_studies_j_chem_educ/featured_hud109d9c66059217af2bab44795aa78ad_52218_550x0_resize_q90_lanczos_2.png alt class=img-responsive></a><div class=card-text><h4><a href=/open-source-toolkit/post/sweepstat_a_buildityourself_twoelectrode_potentiostat_for_macroelectrode_and_ultramicroelectrode_studies_j_chem_educ/>SweepStat: A Build-It-Yourself, Two-Electrode Potentiostat for Macroelectrode and Ultramicroelectrode Studies</a></h4><div class=article-style><p>Publishing date: &lsquo;2020-02-07&rsquo;
Published on: &lsquo;J. Chem. Educ&rsquo;
summary: &lsquo;None&rsquo;
authors: &lsquo;Matthew W. Glasscott, Matthew D. Verber, Jackson R. Hall, Andrew D. Pendergast, Collin J. McKinney, Jeffrey E. Dick&rsquo;
link to paper: &lsquo;https://pubs.</p></div></div></div></div><div class="project-card project-item isotope-item js-id-Research-Article js-id-None js-id-HardwareX"><div class=card><a href=/open-source-toolkit/post/%C2%B5pump_an_opensource_pressure_pump_for_precision_fluid_handling_in_microfluidics_hardwarex/ class="card-image hover-overlay"><img src=/open-source-toolkit/post/%C2%B5pump_an_opensource_pressure_pump_for_precision_fluid_handling_in_microfluidics_hardwarex/featured_hud109d9c66059217af2bab44795aa78ad_52218_550x0_resize_q90_lanczos_2.png alt class=img-responsive></a><div class=card-text><h4><a href=/open-source-toolkit/post/%C2%B5pump_an_opensource_pressure_pump_for_precision_fluid_handling_in_microfluidics_hardwarex/>µPump: An open-source pressure pump for precision fluid handling in microfluidics</a></h4><div class=article-style><p>Publishing date: &lsquo;2020-02-07&rsquo;
Published on: &lsquo;HardwareX&rsquo;
summary: &lsquo;None&rsquo;
authors: &lsquo;Run Ze Gao, Marie Hébert, Jan Huissoon, Carolyn L.Ren&rsquo;
link to paper: &lsquo;https://www.sciencedirect.com/science/article/pii/S2468067220300043'
Icons made by catkuro from www.flaticon.com</p></div></div></div></div></div></div></div></div></section><section id=tags class="home-section wg-tag-cloud"><div class=container><div class=row><div class="col-12 col-lg-4 section-heading"><h1>Popular Topics</h1></div><div class="col-12 col-lg-8"><div class=tag-cloud><a href=/open-source-toolkit/tag/biorxiv/ style=font-size:1.4889323151539333rem>bioRxiv</a>
<a href=/open-source-toolkit/tag/bmc-bioinformatics/ style=font-size:.9280525826565408rem>BMC Bioinformatics</a>
<a href=/open-source-toolkit/tag/commentary/ style=font-size:1.061454791703229rem>Commentary</a>
<a href=/open-source-toolkit/tag/hardware-x/ style=font-size:1.1561051653130816rem>Hardware X</a>
<a href=/open-source-toolkit/tag/hardwarex/ style=font-size:1.061454791703229rem>HardwareX</a>
<a href=/open-source-toolkit/tag/nature/ style=font-size:.9280525826565408rem>Nature</a>
<a href=/open-source-toolkit/tag/none/ style=font-size:1.9936112491630116rem>None</a>
<a href=/open-source-toolkit/tag/plos-biology/ style=font-size:.9280525826565408rem>PLOS Biology</a>
<a href=/open-source-toolkit/tag/plos-computational-biology/ style=font-size:1.4575742814383856rem>PLOS Computational Biology</a>
<a href=/open-source-toolkit/tag/plos-one/ style=font-size:1.517559957016311rem>PLOS ONE</a>
<a href=/open-source-toolkit/tag/research-article/ style=font-size:1.9804490808175772rem>Research Article</a>
<a href=/open-source-toolkit/tag/web-article/ style=font-size:1.517559957016311rem>Web Article</a>
<a href=/open-source-toolkit/tag/1/ style=font-size:.7rem>1</a>
<a href=/open-source-toolkit/tag/2/ style=font-size:.7rem>2</a>
<a href=/open-source-toolkit/tag/3/ style=font-size:.7rem>3</a>
<a href=/open-source-toolkit/tag/4/ style=font-size:.7rem>4</a>
<a href=/open-source-toolkit/tag/5/ style=font-size:.7rem>5</a>
<a href=/open-source-toolkit/tag/10/ style=font-size:.7rem>10</a>
<a href=/open-source-toolkit/tag/11/ style=font-size:.7rem>11</a>
<a href=/open-source-toolkit/tag/12/ style=font-size:.7rem>12</a></div></div></div></div></section><section id=about class="home-section wg-about"><div class=container><div class=row><div class="col-12 col-lg-4"><div id=profile><img class="avatar avatar-circle" src=/open-source-toolkit/authors/admin/avatar_hud109d9c66059217af2bab44795aa78ad_73385_270x270_fill_lanczos_center_2.png alt><div class=portrait-title><h2></h2></div><ul class=network-icon aria-hidden=true><li><a href=/open-source-toolkit/#contact><i class="fas fa-envelope big-icon"></i></a></li><li><a href=https://twitter.com/openneurosci target=_blank rel=noopener><i class="fab fa-twitter big-icon"></i></a></li><li><a href=https://github.com/amchagas/open-source-toolkit target=_blank rel=noopener><i class="fab fa-github big-icon"></i></a></li></ul></div></div><div class="col-12 col-lg-8"><h1>About</h1><p>We are network of collaborators trying to keep track and curate interesting open source projects that have been published as peer reviewed or preprints. If you have a project that you’d like to see listed here or if you know of a project that should be listed, drop us a line, via E-mail, or Twitter.</p><div class=row></div></div></div></div></section><section id=people class="home-section wg-people"><div class=container><div class="row justify-content-center people-widget"><div class=col-md-12><h2 class=mb-4>Team</h2></div><div class="col-12 col-sm-auto people-person"><a href=/open-source-toolkit/authors/andre-maia-chagas/><img class="avatar avatar-circle" src=/open-source-toolkit/authors/andre-maia-chagas/avatar_huf96846f355b91e33f33b3e88ba26eca5_14876_270x270_fill_q90_lanczos_center.jpg alt=Avatar></a><div class=portrait-title><h2><a href=/open-source-toolkit/authors/andre-maia-chagas/></a></h2><h3>André Maia Chagas</h3><ul class=network-icon aria-hidden=true><li><a href=/open-source-toolkit/#contact><i class="fas fa-envelope"></i></a></li><li><a href=https://twitter.com/Chagas_AM target=_blank rel=noopener><i class="fab fa-twitter"></i></a></li><li><a href=https://github.com/amchagas target=_blank rel=noopener><i class="fab fa-github"></i></a></li><li><a href="https://scholar.google.com/citations?user=CGSly_wAAAAJ&hl=en" target=_blank rel=noopener><i class="ai ai-google-scholar"></i></a></li></ul></div></div><div class="col-12 col-sm-auto people-person"><a href=/open-source-toolkit/authors/jenny-molloy/><img class="avatar avatar-circle" src=/open-source-toolkit/authors/jenny-molloy/avatar_hucca8b92f6a585ec31ebb1dcaeaf36b2e_8978_270x270_fill_lanczos_center_2.png alt=Avatar></a><div class=portrait-title><h2><a href=/open-source-toolkit/authors/jenny-molloy/></a></h2><h3>Jenny Molloy</h3><ul class=network-icon aria-hidden=true><li><a href=/open-source-toolkit/#contact><i class="fas fa-envelope"></i></a></li><li><a href=https://twitter.com/Chagas_AM target=_blank rel=noopener><i class="fab fa-twitter"></i></a></li><li><a href=https://github.com/amchagas target=_blank rel=noopener><i class="fab fa-github"></i></a></li><li><a href="https://scholar.google.com/citations?user=CGSly_wAAAAJ&hl=en" target=_blank rel=noopener><i class="ai ai-google-scholar"></i></a></li></ul></div></div><div class="col-12 col-sm-auto people-person"><a href=/open-source-toolkit/authors/tom-baden/><img class="avatar avatar-circle" src=/open-source-toolkit/authors/tom-baden/avatar_hucca8b92f6a585ec31ebb1dcaeaf36b2e_8978_270x270_fill_lanczos_center_2.png alt=Avatar></a><div class=portrait-title><h2><a href=/open-source-toolkit/authors/tom-baden/></a></h2><h3>Tom Baden</h3><ul class=network-icon aria-hidden=true><li><a href=/open-source-toolkit/#contact><i class="fas fa-envelope"></i></a></li><li><a href=https://twitter.com/Chagas_AM target=_blank rel=noopener><i class="fab fa-twitter"></i></a></li><li><a href=https://github.com/amchagas target=_blank rel=noopener><i class="fab fa-github"></i></a></li><li><a href="https://scholar.google.com/citations?user=CGSly_wAAAAJ&hl=en" target=_blank rel=noopener><i class="ai ai-google-scholar"></i></a></li></ul></div></div></div></div></section><section id=contact class="home-section wg-contact"><div class=container><div class="row contact-widget"><div class="col-12 col-lg-4 section-heading"><h1>Contact</h1></div><div class="col-12 col-lg-8"><div class=mb-3><form name=contact method=post action=https://formspree.io/a.maia-chagas@sussex.ac.uk><div class="form-group form-inline"><label class=sr-only for=inputName>Name</label>
<input type=text name=name class="form-control w-100" id=inputName placeholder=Name required></div><div class="form-group form-inline"><label class=sr-only for=inputEmail>Email</label>
<input type=email name=email class="form-control w-100" id=inputEmail placeholder=Email required></div><div class=form-group><label class=sr-only for=inputMessage>Message</label>
<textarea name=message class=form-control id=inputMessage rows=5 placeholder=Message required></textarea></div><button type=submit class="btn btn-outline-primary px-3 py-2">Send</button></form></div><ul class=fa-ul><li><i class="fa-li fab fa-twitter fa-2x" aria-hidden=true></i><a href=https://twitter.com/openneurosci target=_blank rel=noopener>Follow Us</a></li></ul></div></div></div></section><script src=https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.min.js integrity="sha256-9/aliU8dGd2tb6OSsuzixeV4y/faTqgFtohetphbbj0=" crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/jquery.imagesloaded/4.1.4/imagesloaded.pkgd.min.js integrity="sha256-lqvxZrPLtfffUl2G/e7szqSvPBILGbwmsGE1MKlOi0Q=" crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/jquery.isotope/3.0.6/isotope.pkgd.min.js integrity="sha256-CBrpuqrMhXwcLLUd5tvQ4euBHCdh7wGlDfNz8vbu/iI=" crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.5.7/jquery.fancybox.min.js integrity="sha256-yt2kYMy0w8AbtF89WXb2P1rfjcP/HTHLT7097U8Y5b8=" crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/instant.page/5.1.0/instantpage.min.js integrity="sha512-1+qUtKoh9XZW7j+6LhRMAyOrgSQKenQ4mluTR+cvxXjP1Z54RxZuzstR/H9kgPXQsVB8IW7DMDFUJpzLjvhGSQ==" crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.2.0/highlight.min.js integrity="sha512-TDKKr+IvoqZnPzc3l35hdjpHD0m+b2EC2SrLEgKDRWpxf2rFCxemkgvJ5kfU48ip+Y+m2XVKyOCD85ybtlZDmw==" crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.2.0/languages/r.min.js></script><script>const code_highlighting=true;</script><script>const search_config={"indexURI":"/open-source-toolkit/index.json","minLength":1,"threshold":0.3};const i18n={"no_results":"No results found","placeholder":"Search...","results":"results found"};const content_type={'post':"Posts",'project':"Projects",'publication':"Publications",'talk':"Talks",'slides':"Slides"};</script><script id=search-hit-fuse-template type=text/x-template>
      <div class="search-hit" id="summary-{{key}}">
      <div class="search-hit-content">
        <div class="search-hit-name">
          <a href="{{relpermalink}}">{{title}}</a>
          <div class="article-metadata search-hit-type">{{type}}</div>
          <p class="search-hit-description">{{snippet}}</p>
        </div>
      </div>
      </div>
    </script><script src=https://cdnjs.cloudflare.com/ajax/libs/fuse.js/3.2.1/fuse.min.js integrity="sha256-VzgmKYmhsGNNN4Ph1kMW+BjoYJM2jV5i4IlFoeZA9XI=" crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/jquery.mark.min.js integrity="sha256-4HLtjeVgH0eIB3aZ9mLYF6E8oU5chNdjU6p6rrXpl9U=" crossorigin=anonymous></script><script src=/open-source-toolkit/js/wowchemy.min.a184847f5de712f1f8597036386ae9ab.js></script><div class=container><footer class=site-footer><p class=powered-by>CC BY SA 4.0</p><p class="powered-by copyright-license-text">This work is licensed under <a href=https://creativecommons.org/licenses/by-nd/4.0 rel="noopener noreferrer" target=_blank>CC BY ND 4.0</a></p><p class="powered-by footer-license-icons"><a href=https://creativecommons.org/licenses/by-nd/4.0 rel="noopener noreferrer" target=_blank><img src=https://search.creativecommons.org/static/img/cc_icon.svg alt="CC icon">
<img src=https://search.creativecommons.org/static/img/cc-by_icon.svg alt="CC by icon">
<img src=https://search.creativecommons.org/static/img/cc-nd_icon.svg alt="CC ND icon"></a></p><p class=powered-by>Published with
<a href=https://wowchemy.com target=_blank rel=noopener>Wowchemy</a> —
the free, <a href=https://github.com/wowchemy/wowchemy-hugo-modules target=_blank rel=noopener>open source</a> website builder that empowers creators.
<span class=float-right aria-hidden=true><a href=# class=back-to-top><span class=button_icon><i class="fas fa-chevron-up fa-2x"></i></span></a></span></p></footer></div><div id=modal class="modal fade" role=dialog><div class=modal-dialog><div class=modal-content><div class=modal-header><h5 class=modal-title>Cite</h5><button type=button class=close data-dismiss=modal aria-label=Close>
<span aria-hidden=true>&#215;</span></button></div><div class=modal-body><pre><code class="tex hljs"></code></pre></div><div class=modal-footer><a class="btn btn-outline-primary my-1 js-copy-cite" href=# target=_blank><i class="fas fa-copy"></i>Copy</a>
<a class="btn btn-outline-primary my-1 js-download-cite" href=# target=_blank><i class="fas fa-download"></i>Download</a><div id=modal-error></div></div></div></div></div></body></html>