<!doctype html><html lang=en-us><head><meta charset=utf-8><meta name=viewport content="width=device-width,initial-scale=1"><meta http-equiv=x-ua-compatible content="IE=edge"><meta name=generator content="Wowchemy 4.8.0 for Hugo"><meta name=description content="Open Neuroscience is a volunteer organized project to aggregate and curate open source projects related to Neurosciences"><link rel=alternate hreflang=en-us href=https://amchagas.github.io/open-source-toolkit/post/><link rel=preconnect href=https://fonts.gstatic.com crossorigin><meta name=theme-color content="#2962ff"><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.14.0/css/all.min.css integrity="sha256-FMvZuGapsJLjouA6k7Eo2lusoAX9i0ShlWFG6qt7SLc=" crossorigin=anonymous><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.5.7/jquery.fancybox.min.css integrity="sha256-Vzbj7sDDS/woiFS3uNKo8eIuni59rjyNGtXfstRzStA=" crossorigin=anonymous><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.2.0/styles/github.min.css crossorigin=anonymous title=hl-light><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.2.0/styles/dracula.min.css crossorigin=anonymous title=hl-dark disabled><script src=https://cdnjs.cloudflare.com/ajax/libs/lazysizes/5.2.2/lazysizes.min.js integrity="sha512-TmDwFLhg3UA4ZG0Eb4MIyT1O1Mb+Oww5kFG0uHqXsdbyZz9DcvYQhKpGgNkamAI6h2lGGZq2X8ftOJvF/XjTUg==" crossorigin=anonymous async></script><link rel=stylesheet href="https://fonts.googleapis.com/css?family=Montserrat:400,700%7CRoboto:400,400italic,700%7CRoboto+Mono&display=swap"><link rel=stylesheet href=/open-source-toolkit/css/wowchemy.css><link rel=alternate href=/open-source-toolkit/post/index.xml type=application/rss+xml title="Open Source Toolkit"><link rel=manifest href=/open-source-toolkit/index.webmanifest><link rel=icon type=image/png href=/open-source-toolkit/images/icon_hu0b7a4cb9992c9ac0e91bd28ffd38dd00_9727_32x32_fill_lanczos_center_2.png><link rel=apple-touch-icon type=image/png href=/open-source-toolkit/images/icon_hu0b7a4cb9992c9ac0e91bd28ffd38dd00_9727_192x192_fill_lanczos_center_2.png><link rel=canonical href=https://amchagas.github.io/open-source-toolkit/post/><meta property="twitter:card" content="summary_large_image"><meta property="twitter:site" content="@openneurosci"><meta property="twitter:creator" content="@openneurosci"><meta property="og:site_name" content="Open Source Toolkit"><meta property="og:url" content="https://amchagas.github.io/open-source-toolkit/post/"><meta property="og:title" content="Posts | Open Source Toolkit"><meta property="og:description" content="Open Neuroscience is a volunteer organized project to aggregate and curate open source projects related to Neurosciences"><meta property="og:image" content="https://amchagas.github.io/open-source-toolkit/media/openneuroscience_logo_dark.svg"><meta property="twitter:image" content="https://amchagas.github.io/open-source-toolkit/media/openneuroscience_logo_dark.svg"><meta property="og:locale" content="en-us"><meta property="og:updated_time" content="2020-08-18T00:00:00+00:00"><head><link rel=stylesheet href=applause-button.css><script src=applause-button.js></script></head><title>Posts | Open Source Toolkit</title></head><body id=top data-spy=scroll data-offset=70 data-target=#TableOfContents><script>window.wcDarkLightEnabled=true;</script><script>const isSiteThemeDark=false;</script><script src=/open-source-toolkit/js/load-theme.js></script><aside class=search-results id=search><div class=container><section class=search-header><div class="row no-gutters justify-content-between mb-3"><div class=col-6><h1>Search</h1></div><div class="col-6 col-search-close"><a class=js-search href=#><i class="fas fa-times-circle text-muted" aria-hidden=true></i></a></div></div><div id=search-box><input name=q id=search-query placeholder=Search... autocapitalize=off autocomplete=off autocorrect=off spellcheck=false type=search class=form-control></div></section><section class=section-search-results><div id=search-hits></div></section></div></aside><nav class="navbar navbar-expand-lg navbar-light compensate-for-scrollbar" id=navbar-main><div class=container><div class="d-none d-lg-inline-flex"><a class=navbar-brand href=/open-source-toolkit/>Open Source Toolkit</a></div><button type=button class=navbar-toggler data-toggle=collapse data-target=#navbar-content aria-controls=navbar aria-expanded=false aria-label="Toggle navigation">
<span><i class="fas fa-bars"></i></span></button><div class="navbar-brand-mobile-wrapper d-inline-flex d-lg-none"><a class=navbar-brand href=/open-source-toolkit/>Open Source Toolkit</a></div><div class="navbar-collapse main-menu-item collapse justify-content-center" id=navbar-content><ul class="navbar-nav d-md-inline-flex"><li class=nav-item><a class=nav-link href=/open-source-toolkit/#intro><span>Home</span></a></li><li class=nav-item><a class=nav-link href=/open-source-toolkit/#topics><span>Topics</span></a></li><li class=nav-item><a class=nav-link href=/open-source-toolkit/#about><span>About</span></a></li><li class=nav-item><a class=nav-link href=/open-source-toolkit/#people><span>Team</span></a></li><li class=nav-item><a class=nav-link href=/open-source-toolkit/#contact><span>Contact</span></a></li></ul></div><ul class="nav-icons navbar-nav flex-row ml-auto d-flex pl-md-2"><li class=nav-item><a class="nav-link js-search" href=# aria-label=Search><i class="fas fa-search" aria-hidden=true></i></a></li><li class="nav-item dropdown theme-dropdown"><a href=# class=nav-link data-toggle=dropdown aria-haspopup=true><i class="fas fa-moon" aria-hidden=true></i></a><div class=dropdown-menu><a href=# class="dropdown-item js-set-theme-light"><span>Light</span></a>
<a href=# class="dropdown-item js-set-theme-dark"><span>Dark</span></a>
<a href=# class="dropdown-item js-set-theme-auto"><span>Automatic</span></a></div></li></ul></div></nav><div class="universal-wrapper pt-3"><h1>Posts</h1></div><div class=universal-wrapper><div class=card-simple><div class=article-metadata><div></div><span class=article-date>Jan 15, 2020</span>
<span class=middot-divider></span><span class=article-reading-time>1 min read</span>
<span class=middot-divider></span><span class=clap data-url=https://amchagas.github.io/open-source-toolkit/post/gencore_an_efficient_tool_to_generate_consensus_reads_for_error_suppressing_and_duplicate_removing_of_ngs_data__bmc_bioinformatics_/>👏<span class=count></span></span>
<script src=https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.min.js></script><script>function loadClapCount(){var elements=jQuery(".clap").toArray();var urls=elements.map(function(el){return el.getAttribute("data-url");});jQuery.ajax({url:"https://api.applause-button.com/get-multiple",method:"POST",data:JSON.stringify(urls),headers:{"Content-Type":"text/plain"},contentType:"text/plain"}).done(function(claps){jQuery(".clap").each(function(){var elem=jQuery(this),url=elem.attr("data-url").replace(/^https?:\/\//,"");var clapCount=claps.find(function(c){return c.url===url;});if(clapCount&&clapCount.claps>0){elem.css("display","initial").find(".count").html(clapCount.claps);}else{elem.css("display","initial").find(".count").html(0);}});});}
loadClapCount();</script></div><a href=/open-source-toolkit/post/gencore_an_efficient_tool_to_generate_consensus_reads_for_error_suppressing_and_duplicate_removing_of_ngs_data__bmc_bioinformatics_/><img src=/open-source-toolkit/post/gencore_an_efficient_tool_to_generate_consensus_reads_for_error_suppressing_and_duplicate_removing_of_ngs_data__bmc_bioinformatics_/featured_hud109d9c66059217af2bab44795aa78ad_52218_918x517_fill_q90_lanczos_smart1_2.png class=article-banner alt="Gencore: an efficient tool to generate consensus reads for error suppressing and duplicate removing of NGS data"></a><h3 class="article-title mb-1 mt-3"><a href=/open-source-toolkit/post/gencore_an_efficient_tool_to_generate_consensus_reads_for_error_suppressing_and_duplicate_removing_of_ngs_data__bmc_bioinformatics_/>Gencore: an efficient tool to generate consensus reads for error suppressing and duplicate removing of NGS data</a></h3><a href=/open-source-toolkit/post/gencore_an_efficient_tool_to_generate_consensus_reads_for_error_suppressing_and_duplicate_removing_of_ngs_data__bmc_bioinformatics_/ class=summary-link><div class=article-style><p>Publishing date: 2020-01-15
Published on: BMC Bioinformatics
summary: None
authors: Shifu Chen, Yanqing Zhou, Yaru Chen, Tanxiao Huang, Wenting Liao, Yun Xu, Zhicheng Li & Jia Gu
link to paper: https://bmcbioinformatics.</p></div></a></div><div class=card-simple><div class=article-metadata><div></div><span class=article-date>Jan 15, 2020</span>
<span class=middot-divider></span><span class=article-reading-time>1 min read</span>
<span class=middot-divider></span><span class=clap data-url=https://amchagas.github.io/open-source-toolkit/post/ice_contract_with_github_sparks_developer_protests__the_atlantic/>👏<span class=count></span></span>
<script src=https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.min.js></script><script>function loadClapCount(){var elements=jQuery(".clap").toArray();var urls=elements.map(function(el){return el.getAttribute("data-url");});jQuery.ajax({url:"https://api.applause-button.com/get-multiple",method:"POST",data:JSON.stringify(urls),headers:{"Content-Type":"text/plain"},contentType:"text/plain"}).done(function(claps){jQuery(".clap").each(function(){var elem=jQuery(this),url=elem.attr("data-url").replace(/^https?:\/\//,"");var clapCount=claps.find(function(c){return c.url===url;});if(clapCount&&clapCount.claps>0){elem.css("display","initial").find(".count").html(clapCount.claps);}else{elem.css("display","initial").find(".count").html(0);}});});}
loadClapCount();</script></div><a href=/open-source-toolkit/post/ice_contract_with_github_sparks_developer_protests__the_atlantic/><img src=/open-source-toolkit/post/ice_contract_with_github_sparks_developer_protests__the_atlantic/featured_hud109d9c66059217af2bab44795aa78ad_52218_918x517_fill_q90_lanczos_smart1_2.png class=article-banner alt="ICE Contract With GitHub Sparks Developer Protests"></a><h3 class="article-title mb-1 mt-3"><a href=/open-source-toolkit/post/ice_contract_with_github_sparks_developer_protests__the_atlantic/>ICE Contract With GitHub Sparks Developer Protests</a></h3><a href=/open-source-toolkit/post/ice_contract_with_github_sparks_developer_protests__the_atlantic/ class=summary-link><div class=article-style><p>Publishing date: 2020-01-15
Published on: The Atlantic
summary: With 37 million users, GitHub is the largest host of source code in the world. Much of the code hosted on GitHub is open source, meaning it’s accessible, shareable, and modifiable to anyone.</p></div></a></div><div class=card-simple><div class=article-metadata><div></div><span class=article-date>Jan 15, 2020</span>
<span class=middot-divider></span><span class=article-reading-time>1 min read</span>
<span class=middot-divider></span><span class=clap data-url=https://amchagas.github.io/open-source-toolkit/post/nufeb_a_massively_parallel_simulator_for_individualbased_modelling_of_microbial_communities_plos_computational_biology/>👏<span class=count></span></span>
<script src=https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.min.js></script><script>function loadClapCount(){var elements=jQuery(".clap").toArray();var urls=elements.map(function(el){return el.getAttribute("data-url");});jQuery.ajax({url:"https://api.applause-button.com/get-multiple",method:"POST",data:JSON.stringify(urls),headers:{"Content-Type":"text/plain"},contentType:"text/plain"}).done(function(claps){jQuery(".clap").each(function(){var elem=jQuery(this),url=elem.attr("data-url").replace(/^https?:\/\//,"");var clapCount=claps.find(function(c){return c.url===url;});if(clapCount&&clapCount.claps>0){elem.css("display","initial").find(".count").html(clapCount.claps);}else{elem.css("display","initial").find(".count").html(0);}});});}
loadClapCount();</script></div><a href=/open-source-toolkit/post/nufeb_a_massively_parallel_simulator_for_individualbased_modelling_of_microbial_communities_plos_computational_biology/><img src=/open-source-toolkit/post/nufeb_a_massively_parallel_simulator_for_individualbased_modelling_of_microbial_communities_plos_computational_biology/featured_hud109d9c66059217af2bab44795aa78ad_52218_918x517_fill_q90_lanczos_smart1_2.png class=article-banner alt="NUFEB: A massively parallel simulator for individual-based modelling of microbial communities"></a><h3 class="article-title mb-1 mt-3"><a href=/open-source-toolkit/post/nufeb_a_massively_parallel_simulator_for_individualbased_modelling_of_microbial_communities_plos_computational_biology/>NUFEB: A massively parallel simulator for individual-based modelling of microbial communities</a></h3><a href=/open-source-toolkit/post/nufeb_a_massively_parallel_simulator_for_individualbased_modelling_of_microbial_communities_plos_computational_biology/ class=summary-link><div class=article-style><p>Publishing date: 2020-01-15
Published on: PLOS Computational Biology
summary: None
authors: Bowen Li, Denis Taniguchi, Jayathilake Pahala Gedara, Valentina Gogulancea, Rebeca Gonzalez-Cabaleiro, Jinju Chen, Andrew Stephen McGough, Irina Dana Ofiteru, Thomas P.</p></div></a></div><div class=card-simple><div class=article-metadata><div></div><span class=article-date>Jan 15, 2020</span>
<span class=middot-divider></span><span class=article-reading-time>1 min read</span>
<span class=middot-divider></span><span class=clap data-url=https://amchagas.github.io/open-source-toolkit/post/pylandstats_an_opensource_pythonic_library_to_compute_landscape_metrics_plos_one/>👏<span class=count></span></span>
<script src=https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.min.js></script><script>function loadClapCount(){var elements=jQuery(".clap").toArray();var urls=elements.map(function(el){return el.getAttribute("data-url");});jQuery.ajax({url:"https://api.applause-button.com/get-multiple",method:"POST",data:JSON.stringify(urls),headers:{"Content-Type":"text/plain"},contentType:"text/plain"}).done(function(claps){jQuery(".clap").each(function(){var elem=jQuery(this),url=elem.attr("data-url").replace(/^https?:\/\//,"");var clapCount=claps.find(function(c){return c.url===url;});if(clapCount&&clapCount.claps>0){elem.css("display","initial").find(".count").html(clapCount.claps);}else{elem.css("display","initial").find(".count").html(0);}});});}
loadClapCount();</script></div><a href=/open-source-toolkit/post/pylandstats_an_opensource_pythonic_library_to_compute_landscape_metrics_plos_one/><img src=/open-source-toolkit/post/pylandstats_an_opensource_pythonic_library_to_compute_landscape_metrics_plos_one/featured_hud109d9c66059217af2bab44795aa78ad_52218_918x517_fill_q90_lanczos_smart1_2.png class=article-banner alt="PyLandStats: An open-source Pythonic library to compute landscape metrics"></a><h3 class="article-title mb-1 mt-3"><a href=/open-source-toolkit/post/pylandstats_an_opensource_pythonic_library_to_compute_landscape_metrics_plos_one/>PyLandStats: An open-source Pythonic library to compute landscape metrics</a></h3><a href=/open-source-toolkit/post/pylandstats_an_opensource_pythonic_library_to_compute_landscape_metrics_plos_one/ class=summary-link><div class=article-style><p>Publishing date: 2020-01-15
Published on: PLOS ONE
summary: Channel Editor Summary: PyLandStats is a fantastic example of an open source scientific package, with an open licence (GPL), clear documentation for both users and developers, and a source code repository set up to easily enable contributions and bug reports.</p></div></a></div><div class=card-simple><div class=article-metadata><div></div><span class=article-date>Jan 15, 2020</span>
<span class=middot-divider></span><span class=article-reading-time>1 min read</span>
<span class=middot-divider></span><span class=clap data-url=https://amchagas.github.io/open-source-toolkit/post/ten_quick_tips_for_delivering_programming_lessons_plos_computational_biology/>👏<span class=count></span></span>
<script src=https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.min.js></script><script>function loadClapCount(){var elements=jQuery(".clap").toArray();var urls=elements.map(function(el){return el.getAttribute("data-url");});jQuery.ajax({url:"https://api.applause-button.com/get-multiple",method:"POST",data:JSON.stringify(urls),headers:{"Content-Type":"text/plain"},contentType:"text/plain"}).done(function(claps){jQuery(".clap").each(function(){var elem=jQuery(this),url=elem.attr("data-url").replace(/^https?:\/\//,"");var clapCount=claps.find(function(c){return c.url===url;});if(clapCount&&clapCount.claps>0){elem.css("display","initial").find(".count").html(clapCount.claps);}else{elem.css("display","initial").find(".count").html(0);}});});}
loadClapCount();</script></div><a href=/open-source-toolkit/post/ten_quick_tips_for_delivering_programming_lessons_plos_computational_biology/><img src=/open-source-toolkit/post/ten_quick_tips_for_delivering_programming_lessons_plos_computational_biology/featured_hud109d9c66059217af2bab44795aa78ad_52218_918x517_fill_q90_lanczos_smart1_2.png class=article-banner alt="Ten quick tips for delivering programming lessons"></a><h3 class="article-title mb-1 mt-3"><a href=/open-source-toolkit/post/ten_quick_tips_for_delivering_programming_lessons_plos_computational_biology/>Ten quick tips for delivering programming lessons</a></h3><a href=/open-source-toolkit/post/ten_quick_tips_for_delivering_programming_lessons_plos_computational_biology/ class=summary-link><div class=article-style><p>Publishing date: 2020-01-15
Published on: PLOS Computational Biology
summary: Teaching well is a craft like any other, and success often comes from an accumulation of small improvements rather than from any single large change.</p></div></a></div><div class=card-simple><div class=article-metadata><div></div><span class=article-date>Nov 14, 2019</span>
<span class=middot-divider></span><span class=article-reading-time>1 min read</span>
<span class=middot-divider></span><span class=clap data-url=https://amchagas.github.io/open-source-toolkit/post/bionitio_demonstrating_and_facilitating_best_practices_for_bioinformatics_commandline_software_giga_science/>👏<span class=count></span></span>
<script src=https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.min.js></script><script>function loadClapCount(){var elements=jQuery(".clap").toArray();var urls=elements.map(function(el){return el.getAttribute("data-url");});jQuery.ajax({url:"https://api.applause-button.com/get-multiple",method:"POST",data:JSON.stringify(urls),headers:{"Content-Type":"text/plain"},contentType:"text/plain"}).done(function(claps){jQuery(".clap").each(function(){var elem=jQuery(this),url=elem.attr("data-url").replace(/^https?:\/\//,"");var clapCount=claps.find(function(c){return c.url===url;});if(clapCount&&clapCount.claps>0){elem.css("display","initial").find(".count").html(clapCount.claps);}else{elem.css("display","initial").find(".count").html(0);}});});}
loadClapCount();</script></div><a href=/open-source-toolkit/post/bionitio_demonstrating_and_facilitating_best_practices_for_bioinformatics_commandline_software_giga_science/><img src=/open-source-toolkit/post/bionitio_demonstrating_and_facilitating_best_practices_for_bioinformatics_commandline_software_giga_science/featured_hud109d9c66059217af2bab44795aa78ad_52218_918x517_fill_q90_lanczos_smart1_2.png class=article-banner alt="Bionitio: demonstrating and facilitating best practices for bioinformatics command-line software"></a><h3 class="article-title mb-1 mt-3"><a href=/open-source-toolkit/post/bionitio_demonstrating_and_facilitating_best_practices_for_bioinformatics_commandline_software_giga_science/>Bionitio: demonstrating and facilitating best practices for bioinformatics command-line software</a></h3><a href=/open-source-toolkit/post/bionitio_demonstrating_and_facilitating_best_practices_for_bioinformatics_commandline_software_giga_science/ class=summary-link><div class=article-style><p>Publishing date: 2019-11-14
Published on: Giga Science
summary: The authors have developed Bionitio, a tool that automates the process of starting new bioinformatics software projects following recommended best practices. With a single command, the user can create a new well-structured project in 1 of 12 programming languages.</p></div></a></div><div class=card-simple><div class=article-metadata><div></div><span class=article-date>Nov 14, 2019</span>
<span class=middot-divider></span><span class=article-reading-time>1 min read</span>
<span class=middot-divider></span><span class=clap data-url=https://amchagas.github.io/open-source-toolkit/post/deepimpute_an_accurate_fast_and_scalable_deep_neural_network_method_to_impute_singlecell_rnaseq_data_genome_biology/>👏<span class=count></span></span>
<script src=https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.min.js></script><script>function loadClapCount(){var elements=jQuery(".clap").toArray();var urls=elements.map(function(el){return el.getAttribute("data-url");});jQuery.ajax({url:"https://api.applause-button.com/get-multiple",method:"POST",data:JSON.stringify(urls),headers:{"Content-Type":"text/plain"},contentType:"text/plain"}).done(function(claps){jQuery(".clap").each(function(){var elem=jQuery(this),url=elem.attr("data-url").replace(/^https?:\/\//,"");var clapCount=claps.find(function(c){return c.url===url;});if(clapCount&&clapCount.claps>0){elem.css("display","initial").find(".count").html(clapCount.claps);}else{elem.css("display","initial").find(".count").html(0);}});});}
loadClapCount();</script></div><a href=/open-source-toolkit/post/deepimpute_an_accurate_fast_and_scalable_deep_neural_network_method_to_impute_singlecell_rnaseq_data_genome_biology/><img src=/open-source-toolkit/post/deepimpute_an_accurate_fast_and_scalable_deep_neural_network_method_to_impute_singlecell_rnaseq_data_genome_biology/featured_hud109d9c66059217af2bab44795aa78ad_52218_918x517_fill_q90_lanczos_smart1_2.png class=article-banner alt="DeepImpute: an accurate, fast, and scalable deep neural network method to impute single-cell RNA-seq data"></a><h3 class="article-title mb-1 mt-3"><a href=/open-source-toolkit/post/deepimpute_an_accurate_fast_and_scalable_deep_neural_network_method_to_impute_singlecell_rnaseq_data_genome_biology/>DeepImpute: an accurate, fast, and scalable deep neural network method to impute single-cell RNA-seq data</a></h3><a href=/open-source-toolkit/post/deepimpute_an_accurate_fast_and_scalable_deep_neural_network_method_to_impute_singlecell_rnaseq_data_genome_biology/ class=summary-link><div class=article-style><p>Publishing date: 2019-11-14
Published on: Genome Biology
summary: Single-cell RNA sequencing (scRNA-seq) offers new opportunities to study gene expression of tens of thousands of single cells simultaneously. The authors present DeepImpute, a deep neural network-based imputation algorithm that uses dropout layers and loss functions to learn patterns in the data, allowing for accurate imputation.</p></div></a></div><div class=card-simple><div class=article-metadata><div></div><span class=article-date>Nov 14, 2019</span>
<span class=middot-divider></span><span class=article-reading-time>1 min read</span>
<span class=middot-divider></span><span class=clap data-url=https://amchagas.github.io/open-source-toolkit/post/introducing_mozilla_open_leaders_x_mozilla_foundation/>👏<span class=count></span></span>
<script src=https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.min.js></script><script>function loadClapCount(){var elements=jQuery(".clap").toArray();var urls=elements.map(function(el){return el.getAttribute("data-url");});jQuery.ajax({url:"https://api.applause-button.com/get-multiple",method:"POST",data:JSON.stringify(urls),headers:{"Content-Type":"text/plain"},contentType:"text/plain"}).done(function(claps){jQuery(".clap").each(function(){var elem=jQuery(this),url=elem.attr("data-url").replace(/^https?:\/\//,"");var clapCount=claps.find(function(c){return c.url===url;});if(clapCount&&clapCount.claps>0){elem.css("display","initial").find(".count").html(clapCount.claps);}else{elem.css("display","initial").find(".count").html(0);}});});}
loadClapCount();</script></div><a href=/open-source-toolkit/post/introducing_mozilla_open_leaders_x_mozilla_foundation/><img src=/open-source-toolkit/post/introducing_mozilla_open_leaders_x_mozilla_foundation/featured_hud109d9c66059217af2bab44795aa78ad_52218_918x517_fill_q90_lanczos_smart1_2.png class=article-banner alt="Introducing Mozilla Open Leaders X"></a><h3 class="article-title mb-1 mt-3"><a href=/open-source-toolkit/post/introducing_mozilla_open_leaders_x_mozilla_foundation/>Introducing Mozilla Open Leaders X</a></h3><a href=/open-source-toolkit/post/introducing_mozilla_open_leaders_x_mozilla_foundation/ class=summary-link><div class=article-style><p>Publishing date: 2019-11-14
Published on: Mozilla Foundation
summary: Here are the new, community-run Open Leaders programs with diverse audiences that serve as entry-points into the internet health movement. From September through November, these programs have gone through movement building training and program-design mentorship in preparation to launch their own Open Leaders program in 2020.</p></div></a></div><div class=card-simple><div class=article-metadata><div></div><span class=article-date>Nov 14, 2019</span>
<span class=middot-divider></span><span class=article-reading-time>1 min read</span>
<span class=middot-divider></span><span class=clap data-url=https://amchagas.github.io/open-source-toolkit/post/measuring_sustainability_of_seedfunded_earth_science_informatics_projects_plos_one/>👏<span class=count></span></span>
<script src=https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.min.js></script><script>function loadClapCount(){var elements=jQuery(".clap").toArray();var urls=elements.map(function(el){return el.getAttribute("data-url");});jQuery.ajax({url:"https://api.applause-button.com/get-multiple",method:"POST",data:JSON.stringify(urls),headers:{"Content-Type":"text/plain"},contentType:"text/plain"}).done(function(claps){jQuery(".clap").each(function(){var elem=jQuery(this),url=elem.attr("data-url").replace(/^https?:\/\//,"");var clapCount=claps.find(function(c){return c.url===url;});if(clapCount&&clapCount.claps>0){elem.css("display","initial").find(".count").html(clapCount.claps);}else{elem.css("display","initial").find(".count").html(0);}});});}
loadClapCount();</script></div><a href=/open-source-toolkit/post/measuring_sustainability_of_seedfunded_earth_science_informatics_projects_plos_one/><img src=/open-source-toolkit/post/measuring_sustainability_of_seedfunded_earth_science_informatics_projects_plos_one/featured_hud109d9c66059217af2bab44795aa78ad_52218_918x517_fill_q90_lanczos_smart1_2.png class=article-banner alt="Measuring sustainability of seed-funded earth science informatics projects"></a><h3 class="article-title mb-1 mt-3"><a href=/open-source-toolkit/post/measuring_sustainability_of_seedfunded_earth_science_informatics_projects_plos_one/>Measuring sustainability of seed-funded earth science informatics projects</a></h3><a href=/open-source-toolkit/post/measuring_sustainability_of_seedfunded_earth_science_informatics_projects_plos_one/ class=summary-link><div class=article-style><p>Publishing date: 2019-11-14
Published on: PLOS ONE
summary: Short term funding is a common funding model for informatics projects. Funders are interested in maximizing the sustainability and accessibility of the outputs, but there are no commonly accepted practices to do so in the Earth sciences informatics field.</p></div></a></div><div class=card-simple><div class=article-metadata><div></div><span class=article-date>Nov 14, 2019</span>
<span class=middot-divider></span><span class=article-reading-time>1 min read</span>
<span class=middot-divider></span><span class=clap data-url=https://amchagas.github.io/open-source-toolkit/post/odmltables_a_userfriendly_approach_for_managing_metadata_of_neurophysiological_experiments_frontiers_in_neuroinformatics/>👏<span class=count></span></span>
<script src=https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.min.js></script><script>function loadClapCount(){var elements=jQuery(".clap").toArray();var urls=elements.map(function(el){return el.getAttribute("data-url");});jQuery.ajax({url:"https://api.applause-button.com/get-multiple",method:"POST",data:JSON.stringify(urls),headers:{"Content-Type":"text/plain"},contentType:"text/plain"}).done(function(claps){jQuery(".clap").each(function(){var elem=jQuery(this),url=elem.attr("data-url").replace(/^https?:\/\//,"");var clapCount=claps.find(function(c){return c.url===url;});if(clapCount&&clapCount.claps>0){elem.css("display","initial").find(".count").html(clapCount.claps);}else{elem.css("display","initial").find(".count").html(0);}});});}
loadClapCount();</script></div><a href=/open-source-toolkit/post/odmltables_a_userfriendly_approach_for_managing_metadata_of_neurophysiological_experiments_frontiers_in_neuroinformatics/><img src=/open-source-toolkit/post/odmltables_a_userfriendly_approach_for_managing_metadata_of_neurophysiological_experiments_frontiers_in_neuroinformatics/featured_hud109d9c66059217af2bab44795aa78ad_52218_918x517_fill_q90_lanczos_smart1_2.png class=article-banner alt="odMLtables: A User-Friendly Approach for Managing Metadata of Neurophysiological Experiments"></a><h3 class="article-title mb-1 mt-3"><a href=/open-source-toolkit/post/odmltables_a_userfriendly_approach_for_managing_metadata_of_neurophysiological_experiments_frontiers_in_neuroinformatics/>odMLtables: A User-Friendly Approach for Managing Metadata of Neurophysiological Experiments</a></h3><a href=/open-source-toolkit/post/odmltables_a_userfriendly_approach_for_managing_metadata_of_neurophysiological_experiments_frontiers_in_neuroinformatics/ class=summary-link><div class=article-style><p>Publishing date: 2019-11-14
Published on: Frontiers in Neuroinformatics
summary: None
authors: Julia Sprenger, Lyuba Zehl, Jana Pick, Michael Sonntag, Jan Grewe, Thomas Wachtler, Sonja Grün, and Michael Denker
link to paper: https://doi.</p></div></a></div><nav><ul class="pagination justify-content-center"><li class=page-item><a class=page-link href=/open-source-toolkit/post/page/9/>&#171;</a></li><li class=page-item><a class=page-link href=/open-source-toolkit/post/page/11/>&#187;</a></li></ul></nav></div><script src=https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.min.js integrity="sha256-9/aliU8dGd2tb6OSsuzixeV4y/faTqgFtohetphbbj0=" crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/jquery.imagesloaded/4.1.4/imagesloaded.pkgd.min.js integrity="sha256-lqvxZrPLtfffUl2G/e7szqSvPBILGbwmsGE1MKlOi0Q=" crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/jquery.isotope/3.0.6/isotope.pkgd.min.js integrity="sha256-CBrpuqrMhXwcLLUd5tvQ4euBHCdh7wGlDfNz8vbu/iI=" crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.5.7/jquery.fancybox.min.js integrity="sha256-yt2kYMy0w8AbtF89WXb2P1rfjcP/HTHLT7097U8Y5b8=" crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/instant.page/5.1.0/instantpage.min.js integrity="sha512-1+qUtKoh9XZW7j+6LhRMAyOrgSQKenQ4mluTR+cvxXjP1Z54RxZuzstR/H9kgPXQsVB8IW7DMDFUJpzLjvhGSQ==" crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.2.0/highlight.min.js integrity="sha512-TDKKr+IvoqZnPzc3l35hdjpHD0m+b2EC2SrLEgKDRWpxf2rFCxemkgvJ5kfU48ip+Y+m2XVKyOCD85ybtlZDmw==" crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.2.0/languages/r.min.js></script><script>const code_highlighting=true;</script><script>const search_config={"indexURI":"/open-source-toolkit/index.json","minLength":1,"threshold":0.3};const i18n={"no_results":"No results found","placeholder":"Search...","results":"results found"};const content_type={'post':"Posts",'project':"Projects",'publication':"Publications",'talk':"Talks",'slides':"Slides"};</script><script id=search-hit-fuse-template type=text/x-template>
      <div class="search-hit" id="summary-{{key}}">
      <div class="search-hit-content">
        <div class="search-hit-name">
          <a href="{{relpermalink}}">{{title}}</a>
          <div class="article-metadata search-hit-type">{{type}}</div>
          <p class="search-hit-description">{{snippet}}</p>
        </div>
      </div>
      </div>
    </script><script src=https://cdnjs.cloudflare.com/ajax/libs/fuse.js/3.2.1/fuse.min.js integrity="sha256-VzgmKYmhsGNNN4Ph1kMW+BjoYJM2jV5i4IlFoeZA9XI=" crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/jquery.mark.min.js integrity="sha256-4HLtjeVgH0eIB3aZ9mLYF6E8oU5chNdjU6p6rrXpl9U=" crossorigin=anonymous></script><script src=/open-source-toolkit/js/wowchemy.min.a184847f5de712f1f8597036386ae9ab.js></script><div class=container><footer class=site-footer><p class=powered-by>CC BY SA 4.0</p><p class="powered-by copyright-license-text">This work is licensed under <a href=https://creativecommons.org/licenses/by-nd/4.0 rel="noopener noreferrer" target=_blank>CC BY ND 4.0</a></p><p class="powered-by footer-license-icons"><a href=https://creativecommons.org/licenses/by-nd/4.0 rel="noopener noreferrer" target=_blank><img src=https://search.creativecommons.org/static/img/cc_icon.svg alt="CC icon">
<img src=https://search.creativecommons.org/static/img/cc-by_icon.svg alt="CC by icon">
<img src=https://search.creativecommons.org/static/img/cc-nd_icon.svg alt="CC ND icon"></a></p><p class=powered-by>Published with
<a href=https://wowchemy.com target=_blank rel=noopener>Wowchemy</a> —
the free, <a href=https://github.com/wowchemy/wowchemy-hugo-modules target=_blank rel=noopener>open source</a> website builder that empowers creators.
<span class=float-right aria-hidden=true><a href=# class=back-to-top><span class=button_icon><i class="fas fa-chevron-up fa-2x"></i></span></a></span></p></footer></div><div id=modal class="modal fade" role=dialog><div class=modal-dialog><div class=modal-content><div class=modal-header><h5 class=modal-title>Cite</h5><button type=button class=close data-dismiss=modal aria-label=Close>
<span aria-hidden=true>&#215;</span></button></div><div class=modal-body><pre><code class="tex hljs"></code></pre></div><div class=modal-footer><a class="btn btn-outline-primary my-1 js-copy-cite" href=# target=_blank><i class="fas fa-copy"></i>Copy</a>
<a class="btn btn-outline-primary my-1 js-download-cite" href=# target=_blank><i class="fas fa-download"></i>Download</a><div id=modal-error></div></div></div></div></div></body></html>